diff -ruN xscreensaver-4.21./driver/XScreenSaver.ad.in xscreensaver-4.21/driver/XScreenSaver.ad.in
--- xscreensaver-4.21./driver/XScreenSaver.ad.in	2005-03-21 06:06:35.000000000 +0100
+++ xscreensaver-4.21/driver/XScreenSaver.ad.in	2005-05-17 18:29:30.945724152 +0200
@@ -94,15 +94,15 @@
 !
 !   Gnome 2.4, 2.6: (yelp can't display man pages, as of 2.6.3)
 !
-@GNOME24@*loadURL: gnome-open '%s'
-@GNOME24@*manualCommand: gnome-terminal --title '%s manual' \
-@GNOME24@		--command '/bin/sh -c "man %s; read foo"'
+!@GNOME24@*loadURL: gnome-open '%s'
+!@GNOME24@*manualCommand: gnome-terminal --title '%s manual' \
+!@GNOME24@		--command '/bin/sh -c "man %s; read foo"'
 !
 !   Gnome 2.2:
 !
-@GNOME22@*loadURL: gnome-url-show '%s'
-@GNOME22@*manualCommand: gnome-terminal --title '%s manual' \
-@GNOME22@		--command '/bin/sh -c "man %s; read foo"'
+!@GNOME22@*loadURL: gnome-url-show '%s'
+!@GNOME22@*manualCommand: gnome-terminal --title '%s manual' \
+!@GNOME22@		--command '/bin/sh -c "man %s; read foo"'
 !
 !   Gnome 1.4:
 !
@@ -111,9 +111,9 @@
 !
 !   non-Gnome systems:
 !
-@NOGNOME@*loadURL: firefox '%s' || mozilla '%s' || netscape '%s'
-@NOGNOME@*manualCommand: xterm -sb -fg black -bg gray75 -T '%s manual' \
-@NOGNOME@		    -e /bin/sh -c 'man "%s" ; read foo'
+*loadURL: firefox '%s' || mozilla '%s' || netscape '%s'
+*manualCommand: xterm -sb -fg black -bg gray75 -T '%s manual' \
+		    -e /bin/sh -c 'man "%s" ; read foo'
 
 
 ! The format used for printing the date and time in the password dialog box
