diff -ur xscreensaver-4.21.orig/hacks/anemone.man xscreensaver-4.21/hacks/anemone.man
--- xscreensaver-4.21.orig/hacks/anemone.man	2002-11-20 08:54:02.000000000 +0100
+++ xscreensaver-4.21/hacks/anemone.man	2005-05-17 17:20:10.000000000 +0200
@@ -1,8 +1,8 @@
 .TH XScreenSaver 1 "" "X Version 11"
 .SH NAME
-anemone - wiggling tentacles.
+anemone - wiggling tentacles
 .SH SYNOPSIS
-.B anemone
+.B /usr/lib/xscreensaver/anemone
 [\-display \fIhost:display.screen\fP]
 [\-visual \fIvisual\fP]
 [\-window]
diff -ur xscreensaver-4.21.orig/hacks/anemotaxis.man xscreensaver-4.21/hacks/anemotaxis.man
--- xscreensaver-4.21.orig/hacks/anemotaxis.man	2004-07-12 21:00:44.000000000 +0200
+++ xscreensaver-4.21/hacks/anemotaxis.man	2005-05-17 17:20:13.000000000 +0200
@@ -1,8 +1,8 @@
 .TH XScreenSaver 1 "" "X Version 11"
 .SH NAME
-anemotaxis - directional search on a plane.
+anemotaxis - directional search on a plane
 .SH SYNOPSIS
-.B anemotaxis
+.B /usr/lib/xscreensaver/anemotaxis
 [\-display \fIhost:display.screen\fP]
 [\-visual \fIvisual\fP]
 [\-window]
diff -ur xscreensaver-4.21.orig/hacks/ant.man xscreensaver-4.21/hacks/ant.man
--- xscreensaver-4.21.orig/hacks/ant.man	2002-11-20 08:54:02.000000000 +0100
+++ xscreensaver-4.21/hacks/ant.man	2005-05-17 17:20:16.000000000 +0200
@@ -1,8 +1,8 @@
 .TH XScreenSaver 1 "" "X Version 11"
 .SH NAME
-ant - cellular automaton.
+ant - cellular automaton
 .SH SYNOPSIS
-.B ant
+.B /usr/lib/xscreensaver/ant
 [\-display \fIhost:display.screen\fP]
 [\-visual \fIvisual\fP]
 [\-window]
diff -ur xscreensaver-4.21.orig/hacks/apollonian.man xscreensaver-4.21/hacks/apollonian.man
--- xscreensaver-4.21.orig/hacks/apollonian.man	2002-05-31 05:07:18.000000000 +0200
+++ xscreensaver-4.21/hacks/apollonian.man	2005-05-17 17:20:22.000000000 +0200
@@ -1,8 +1,8 @@
 .TH XScreenSaver 1 "" "X Version 11"
 .SH NAME
-apollonian - Descartes Circle Theorem.
+apollonian - Descartes Circle Theorem
 .SH SYNOPSIS
-.B apollonian
+.B /usr/lib/xscreensaver/apollonian
 [\-display \fIhost:display.screen\fP]
 [\-visual \fIvisual\fP]
 [\-window]
diff -ur xscreensaver-4.21.orig/hacks/apple2.man xscreensaver-4.21/hacks/apple2.man
--- xscreensaver-4.21.orig/hacks/apple2.man	2005-03-20 21:22:34.000000000 +0100
+++ xscreensaver-4.21/hacks/apple2.man	2005-05-17 17:01:07.000000000 +0200
@@ -15,7 +15,7 @@
 .SH NAME
 apple2 - Apple ][ display emulator
 .SH SYNOPSIS
-.B apple2
+.B /usr/lib/xscreensaver/apple2
 [\-display \fIhost:display.screen\fP] [\-foreground \fIcolor\fP]
 [\-background \fIcolor\fP] [\-window] [\-root] [\-mono] [\-install]
 [\-visual \fIvisual\fP]
diff -ur xscreensaver-4.21.orig/hacks/attraction.man xscreensaver-4.21/hacks/attraction.man
--- xscreensaver-4.21.orig/hacks/attraction.man	2001-05-19 07:13:12.000000000 +0200
+++ xscreensaver-4.21/hacks/attraction.man	2005-05-17 17:01:16.000000000 +0200
@@ -15,7 +15,7 @@
 .SH NAME
 attraction - interactions of opposing forces
 .SH SYNOPSIS
-.B attraction
+.B /usr/lib/xscreensaver/attraction
 [\-display \fIhost:display.screen\fP] [\-foreground \fIcolor\fP]
 [\-background \fIcolor\fP] [\-window] [\-root] [\-mono] [\-install]
 [\-visual \fIvisual\fP] [\-points \fIint\fP] [\-threshold \fIint\fP]
diff -ur xscreensaver-4.21.orig/hacks/barcode.man xscreensaver-4.21/hacks/barcode.man
--- xscreensaver-4.21.orig/hacks/barcode.man	2003-02-08 12:38:23.000000000 +0100
+++ xscreensaver-4.21/hacks/barcode.man	2005-05-17 17:01:20.000000000 +0200
@@ -2,7 +2,7 @@
 .SH NAME
 barcode - draws a random sequence of barcodes for the products you enjoy
 .SH SYNOPSIS
-.B barcode
+.B /usr/lib/xscreensaver/barcode
 [\-display \fIhost:display.screen\fP]
 [\-visual \fIvisual\fP]
 [\-window]
diff -ur xscreensaver-4.21.orig/hacks/blaster.man xscreensaver-4.21/hacks/blaster.man
--- xscreensaver-4.21.orig/hacks/blaster.man	2002-05-31 05:05:26.000000000 +0200
+++ xscreensaver-4.21/hacks/blaster.man	2005-05-17 17:01:24.000000000 +0200
@@ -2,7 +2,7 @@
 .SH NAME
 blaster - simulation of space combat
 .SH SYNOPSIS
-.B blaster
+.B /usr/lib/xscreensaver/blaster
 [\-display \fIhost:display.screen\fP]
 [\-visual \fIvisual\fP]
 [\-window]
diff -ur xscreensaver-4.21.orig/hacks/blitspin.man xscreensaver-4.21/hacks/blitspin.man
--- xscreensaver-4.21.orig/hacks/blitspin.man	2001-04-15 10:55:40.000000000 +0200
+++ xscreensaver-4.21/hacks/blitspin.man	2005-05-17 17:01:33.000000000 +0200
@@ -2,7 +2,7 @@
 .SH NAME
 blitspin - rotate a bitmap in an interesting way
 .SH SYNOPSIS
-.B blitspin
+.B /usr/lib/xscreensaver/blitspin
 [\-display \fIhost:display.screen\fP]
 [\-foreground \fIcolor\fP] [\-background \fIcolor\fP] [\-window] [\-root]
 [\-mono] [\-install] [\-visual \fIvisual\fP] [\-bitmap \fIfilename\fP]
diff -ur xscreensaver-4.21.orig/hacks/bouboule.man xscreensaver-4.21/hacks/bouboule.man
--- xscreensaver-4.21.orig/hacks/bouboule.man	1998-08-03 10:04:56.000000000 +0200
+++ xscreensaver-4.21/hacks/bouboule.man	2005-05-17 17:01:36.000000000 +0200
@@ -2,7 +2,7 @@
 .SH NAME
 bouboule - draws spinning 3D blobs
 .SH SYNOPSIS
-.B bouboule
+.B /usr/lib/xscreensaver/bouboule
 [\-display \fIhost:display.screen\fP] [\-foreground \fIcolor\fP] [\-background \fIcolor\fP] [\-window] [\-root] [\-mono] [\-install] [\-visual \fIvisual\fP] [\-ncolors \fIinteger\fP] [\-delay \fImicroseconds\fP] [\-cycles \fIinteger\fP] [\-count \fIinteger\fP] [\-3d]
 
 .SH DESCRIPTION
diff -ur xscreensaver-4.21.orig/hacks/boxfit.man xscreensaver-4.21/hacks/boxfit.man
--- xscreensaver-4.21.orig/hacks/boxfit.man	2005-02-24 11:10:21.000000000 +0100
+++ xscreensaver-4.21/hacks/boxfit.man	2005-05-17 17:20:41.000000000 +0200
@@ -1,8 +1,8 @@
 .TH XScreenSaver 1 "" "X Version 11"
 .SH NAME
-boxfit - fills space with a gradient of growing boxes or circles.
+boxfit - fills space with a gradient of growing boxes or circles
 .SH SYNOPSIS
-.B boxfit
+.B /usr/lib/xscreensaver/boxfit
 [\-display \fIhost:display.screen\fP]
 [\-visual \fIvisual\fP]
 [\-window]
diff -ur xscreensaver-4.21.orig/hacks/braid.man xscreensaver-4.21/hacks/braid.man
--- xscreensaver-4.21.orig/hacks/braid.man	1998-08-03 10:05:12.000000000 +0200
+++ xscreensaver-4.21/hacks/braid.man	2005-05-17 17:01:43.000000000 +0200
@@ -2,7 +2,7 @@
 .SH NAME
 braid - draws random color-cycling braids around a circle
 .SH SYNOPSIS
-.B braid
+.B /usr/lib/xscreensaver/braid
 [\-display \fIhost:display.screen\fP] [\-foreground \fIcolor\fP] [\-background \fIcolor\fP] [\-window] [\-root] [\-mono] [\-install] [\-visual \fIvisual\fP] [\-ncolors \fIinteger\fP] [\-delay \fImicroseconds\fP] [\-cycles \fIinteger\fP] [\-count \fIinteger\fP]
 
 .SH DESCRIPTION
diff -ur xscreensaver-4.21.orig/hacks/bsod.man xscreensaver-4.21/hacks/bsod.man
--- xscreensaver-4.21.orig/hacks/bsod.man	2005-01-19 04:25:29.000000000 +0100
+++ xscreensaver-4.21/hacks/bsod.man	2005-05-17 17:01:47.000000000 +0200
@@ -15,7 +15,7 @@
 .SH NAME
 bsod - Blue Screen of Death emulator
 .SH SYNOPSIS
-.B bsod
+.B /usr/lib/xscreensaver/bsod
 [\-display \fIhost:display.screen\fP] [\-foreground \fIcolor\fP]
 [\-background \fIcolor\fP] [\-window] [\-root] [\-mono] [\-install]
 [\-visual \fIvisual\fP] [\-delay \fIseconds\fP]
diff -ur xscreensaver-4.21.orig/hacks/bubbles.man xscreensaver-4.21/hacks/bubbles.man
--- xscreensaver-4.21.orig/hacks/bubbles.man	2003-07-12 10:59:55.000000000 +0200
+++ xscreensaver-4.21/hacks/bubbles.man	2005-05-17 17:01:52.000000000 +0200
@@ -15,7 +15,7 @@
 .SH NAME
 bubbles - frying pan / soft drink simulation
 .SH SYNOPSIS
-.B bubbles
+.B /usr/lib/xscreensaver/bubbles
 [\-display \fIhost:display.screen\fP] [\-foreground \fIcolor\fP] [\-background \fIcolor\fP] [\-window] [\-root] [\-mono] [\-install] [\-visual \fIvisual\fP] [\-simple] [\-broken] [\-3D]
 .SH DESCRIPTION
 \fIBubbles\fP sprays lots of little random bubbles all over the window which
diff -ur xscreensaver-4.21.orig/hacks/bumps.man xscreensaver-4.21/hacks/bumps.man
--- xscreensaver-4.21.orig/hacks/bumps.man	2001-04-15 02:24:19.000000000 +0200
+++ xscreensaver-4.21/hacks/bumps.man	2005-05-17 17:02:01.000000000 +0200
@@ -2,7 +2,7 @@
 .SH NAME
 bumps - move distorting spotlight around desktop
 .SH SYNOPSIS
-.B bumps
+.B /usr/lib/xscreensaver/bumps
 [\-display \fIhost:display.screen\fP] [\-foreground \fIcolor\fP]
 [\-background \fIcolor\fP] [\-window] [\-root] [\-mono] [\-install]
 [\-visual \fIvisual\fP] [\-delay \fIusecs\fP] [\-radius \fIpixels\fP]
diff -ur xscreensaver-4.21.orig/hacks/ccurve.man xscreensaver-4.21/hacks/ccurve.man
--- xscreensaver-4.21.orig/hacks/ccurve.man	2002-05-31 05:04:59.000000000 +0200
+++ xscreensaver-4.21/hacks/ccurve.man	2005-05-17 17:21:00.000000000 +0200
@@ -1,8 +1,8 @@
 .TH XScreenSaver 1 "" "X Version 11"
 .SH NAME
-ccurve - self-similar linear fractals.
+ccurve - self-similar linear fractals
 .SH SYNOPSIS
-.B ccurve
+.B /usr/lib/xscreensaver/ccurve
 [\-display \fIhost:display.screen\fP]
 [\-visual \fIvisual\fP]
 [\-window]
diff -ur xscreensaver-4.21.orig/hacks/cloudlife.man xscreensaver-4.21/hacks/cloudlife.man
--- xscreensaver-4.21.orig/hacks/cloudlife.man	2003-06-01 03:34:48.000000000 +0200
+++ xscreensaver-4.21/hacks/cloudlife.man	2005-05-17 17:21:07.000000000 +0200
@@ -1,8 +1,8 @@
-.TH XScreenSaver 6 "20-May-2003" "X Version 11"
+.TH XScreenSaver 1 "20-May-2003" "X Version 11"
 .SH NAME
 cloudlife - a cellular automaton based on Conway's Life
 .SH SYNOPSIS
-.B cloudlife
+.B /usr/lib/xscreensaver/cloudlife
 [\-display \fIhost:display.screen\fP] [\-foreground \fIcolor\fP] [\-background \fIcolor\fP] [\-window] [\-root] [\-mono] [\-install] [\-visual \fIvisual\fP] [\-ncolors \fIinteger\fP] [\-cycle-delay \fImicroseconds\fP] [\-cycle-colors \fIinteger\fP][\-cell-size \fIinteger\fP] [\-initial-density \fIinteger\fP] [\-max-age \fIinteger\fP]
 
 .SH DESCRIPTION
diff -ur xscreensaver-4.21.orig/hacks/compass.man xscreensaver-4.21/hacks/compass.man
--- xscreensaver-4.21.orig/hacks/compass.man	2002-05-31 05:03:07.000000000 +0200
+++ xscreensaver-4.21/hacks/compass.man	2005-05-17 17:21:12.000000000 +0200
@@ -1,8 +1,8 @@
 .TH XScreenSaver 1 "" "X Version 11"
 .SH NAME
-compass - draws a spinning compass.
+compass - draws a spinning compass
 .SH SYNOPSIS
-.B compass
+.B /usr/lib/xscreensaver/compass
 [\-display \fIhost:display.screen\fP]
 [\-visual \fIvisual\fP]
 [\-window]
diff -ur xscreensaver-4.21.orig/hacks/coral.man xscreensaver-4.21/hacks/coral.man
--- xscreensaver-4.21.orig/hacks/coral.man	2002-05-31 04:48:46.000000000 +0200
+++ xscreensaver-4.21/hacks/coral.man	2005-05-17 17:21:15.000000000 +0200
@@ -1,8 +1,8 @@
 .TH XScreenSaver 1 "" "X Version 11"
 .SH NAME
-coral - simulates coral growth, albeit somewhat slowly.
+coral - simulates coral growth, albeit somewhat slowly
 .SH SYNOPSIS
-.B coral
+.B /usr/lib/xscreensaver/coral
 [\-display \fIhost:display.screen\fP]
 [\-visual \fIvisual\fP]
 [\-window]
diff -ur xscreensaver-4.21.orig/hacks/critical.man xscreensaver-4.21/hacks/critical.man
--- xscreensaver-4.21.orig/hacks/critical.man	2000-02-28 05:50:59.000000000 +0100
+++ xscreensaver-4.21/hacks/critical.man	2005-05-17 17:21:19.000000000 +0200
@@ -1,8 +1,8 @@
 .TH XScreenSaver 1 "08 Feb 2000" "X Version 11"
 .SH NAME
-critical - Draw a system showing self-organizing criticality
+critical - draw a system showing self-organizing criticality
 .SH SYNOPSIS
-.B critical
+.B /usr/lib/xscreensaver/critical
 [\-display \fIhost:display.screen\fP] [\-foreground \fIcolor\fP] [\-background \fIcolor\fP] [\-window] [\-root] [\-mono] [\-install] [\-visual \fIvisual\fP] [\-delay \fIseconds\fP] [\-random \fIboolean\fP] [\-ncolors \fIint\fP] [\-offset \fIint\fP] 
 .SH DESCRIPTION
 The \fIcritical\fP program displays a self-organizing critical system
diff -ur xscreensaver-4.21.orig/hacks/crystal.man xscreensaver-4.21/hacks/crystal.man
--- xscreensaver-4.21.orig/hacks/crystal.man	2002-05-31 04:59:40.000000000 +0200
+++ xscreensaver-4.21/hacks/crystal.man	2005-05-17 17:21:23.000000000 +0200
@@ -1,8 +1,8 @@
 .TH XScreenSaver 1 "" "X Version 11"
 .SH NAME
-crystal - kaleidescope.
+crystal - kaleidescope
 .SH SYNOPSIS
-.B crystal
+.B /usr/lib/xscreensaver/crystal
 [\-display \fIhost:display.screen\fP]
 [\-visual \fIvisual\fP]
 [\-window]
diff -ur xscreensaver-4.21.orig/hacks/cynosure.man xscreensaver-4.21/hacks/cynosure.man
--- xscreensaver-4.21.orig/hacks/cynosure.man	2002-05-31 04:55:21.000000000 +0200
+++ xscreensaver-4.21/hacks/cynosure.man	2005-05-17 17:21:25.000000000 +0200
@@ -1,8 +1,8 @@
 .TH XScreenSaver 1 "" "X Version 11"
 .SH NAME
-cynosure - gentle overlapping squares screen saver.
+cynosure - gentle overlapping squares screen saver
 .SH SYNOPSIS
-.B cynosure
+.B /usr/lib/xscreensaver/cynosure
 [\-display \fIhost:display.screen\fP]
 [\-visual \fIvisual\fP]
 [\-window]
diff -ur xscreensaver-4.21.orig/hacks/decayscreen.man xscreensaver-4.21/hacks/decayscreen.man
--- xscreensaver-4.21.orig/hacks/decayscreen.man	2003-02-05 11:05:33.000000000 +0100
+++ xscreensaver-4.21/hacks/decayscreen.man	2005-05-17 17:21:28.000000000 +0200
@@ -1,8 +1,8 @@
 .TH XScreenSaver 1 "05-Apr-1999" "X Version 11"
 .SH NAME
-decayscreen - make a screen meltdown.
+decayscreen - make a screen meltdown
 .SH SYNOPSIS
-.B decayscreen
+.B /usr/lib/xscreensaver/decayscreen
 [\-display \fIhost:display.screen\fP] [\-window] [\-root] [\-mono] [\-install]
 [\-visual \fIvisual\fP] [\-delay \fIusecs\fP] [\-mode \fImode\fP]
 .SH DESCRIPTION
diff -ur xscreensaver-4.21.orig/hacks/deco.man xscreensaver-4.21/hacks/deco.man
--- xscreensaver-4.21.orig/hacks/deco.man	1998-08-03 10:04:48.000000000 +0200
+++ xscreensaver-4.21/hacks/deco.man	2005-05-17 17:02:43.000000000 +0200
@@ -2,7 +2,7 @@
 .SH NAME
 deco - draw tacky 70s basement wall panelling
 .SH SYNOPSIS
-.B deco
+.B /usr/lib/xscreensaver/deco
 [\-display \fIhost:display.screen\fP] [\-foreground \fIcolor\fP] [\-background \fIcolor\fP] [\-window] [\-root] [\-mono] [\-install] [\-visual \fIvisual\fP] [\-delay \fIseconds\fP] [\-max\-depth \fIint\fP] [\-min\-width \fIint\fP] [\-min\-height \fIint\fP] [\-cycle] [\-no\-cycle] [\-cycle\-delay]
 .SH DESCRIPTION
 The \fIdeco\fP program subdivides and colors rectangles randomly.
diff -ur xscreensaver-4.21.orig/hacks/deluxe.man xscreensaver-4.21/hacks/deluxe.man
--- xscreensaver-4.21.orig/hacks/deluxe.man	2002-05-31 05:02:48.000000000 +0200
+++ xscreensaver-4.21/hacks/deluxe.man	2005-05-17 17:21:35.000000000 +0200
@@ -1,8 +1,8 @@
 .TH XScreenSaver 1 "" "X Version 11"
 .SH NAME
-deluxe - pulsing sequence of stars, circles, and lines.
+deluxe - pulsing sequence of stars, circles, and lines
 .SH SYNOPSIS
-.B deluxe
+.B /usr/lib/xscreensaver/deluxe
 [\-display \fIhost:display.screen\fP]
 [\-visual \fIvisual\fP]
 [\-window]
diff -ur xscreensaver-4.21.orig/hacks/demon.man xscreensaver-4.21/hacks/demon.man
--- xscreensaver-4.21.orig/hacks/demon.man	2002-05-31 05:01:17.000000000 +0200
+++ xscreensaver-4.21/hacks/demon.man	2005-05-17 17:21:38.000000000 +0200
@@ -1,8 +1,8 @@
 .TH XScreenSaver 1 "" "X Version 11"
 .SH NAME
-demon - celular automaton.
+demon - celular automaton
 .SH SYNOPSIS
-.B demon
+.B /usr/lib/xscreensaver/demon
 [\-display \fIhost:display.screen\fP]
 [\-visual \fIvisual\fP]
 [\-window]
diff -ur xscreensaver-4.21.orig/hacks/discrete.man xscreensaver-4.21/hacks/discrete.man
--- xscreensaver-4.21.orig/hacks/discrete.man	2002-05-31 05:00:14.000000000 +0200
+++ xscreensaver-4.21/hacks/discrete.man	2005-05-17 17:21:41.000000000 +0200
@@ -1,8 +1,8 @@
 .TH XScreenSaver 1 "" "X Version 11"
 .SH NAME
-discrete - discrete map iterative function fractal systems.
+discrete - discrete map iterative function fractal systems
 .SH SYNOPSIS
-.B discrete
+.B /usr/lib/xscreensaver/discrete
 [\-display \fIhost:display.screen\fP]
 [\-visual \fIvisual\fP]
 [\-window]
diff -ur xscreensaver-4.21.orig/hacks/distort.man xscreensaver-4.21/hacks/distort.man
--- xscreensaver-4.21.orig/hacks/distort.man	2002-09-23 02:58:52.000000000 +0200
+++ xscreensaver-4.21/hacks/distort.man	2005-05-17 17:03:02.000000000 +0200
@@ -2,7 +2,7 @@
 .SH NAME
 distort \- distort the content of the screen in interesting ways
 .SH SYNOPSIS
-.B distort
+.B /usr/lib/xscreensaver/distort
 [\-root] [\-window] [\-mono] [\-install] [\-noinstall] [\-visual \fIvisual\fP]
 [\-window\-id \fIwindow\-id\fP] [\-delay \fIusecs\fP] [\-radius \fIpixels\fP]
 [\-speed \fIint\fP] [\-number \fIint\fP] [\-swamp] [\-bounce] [\-reflect]
diff -ur xscreensaver-4.21.orig/hacks/drift.man xscreensaver-4.21/hacks/drift.man
--- xscreensaver-4.21.orig/hacks/drift.man	1998-08-03 10:05:12.000000000 +0200
+++ xscreensaver-4.21/hacks/drift.man	2005-05-17 17:03:05.000000000 +0200
@@ -2,7 +2,7 @@
 .SH NAME
 drift - draws drifting recursive fractal cosmic flames
 .SH SYNOPSIS
-.B drift
+.B /usr/lib/xscreensaver/drift
 [\-display \fIhost:display.screen\fP] [\-foreground \fIcolor\fP] [\-background \fIcolor\fP] [\-window] [\-root] [\-mono] [\-install] [\-visual \fIvisual\fP] [\-ncolors \fIinteger\fP] [\-delay \fImicroseconds\fP] [\-count \fIinteger\fP] [\-grow] [\-no\-grow] [\-liss] [\-no\-liss]
 
 .SH DESCRIPTION
diff -ur xscreensaver-4.21.orig/hacks/epicycle.man xscreensaver-4.21/hacks/epicycle.man
--- xscreensaver-4.21.orig/hacks/epicycle.man	1998-07-31 03:39:11.000000000 +0200
+++ xscreensaver-4.21/hacks/epicycle.man	2005-05-17 17:03:09.000000000 +0200
@@ -15,7 +15,7 @@
 .SH NAME
 epicycle - draws a point moving around a circle which moves around a cicle which...
 .SH SYNOPSIS
-.B epicycle 
+.B /usr/lib/xscreensaver/epicycle 
 [\-display \fIhost:display.screen\fP] [\-root] [\-window] [\-mono] [\-install] [\-noinstall] [\-visual \fIviz\fP] [\-colors \fIN\fP] [\-foreground \fIname\fP] [\-color\-shift \fIN\fP] [\-delay \fImicroseconds\fP] [\-holdtime \fIseconds\fP] [\-linewidth \fIN\fP] [\-min_circles \fIN\fP] [\-max_circles \fIN\fP] [\-min_speed \fInumber\fP] [\-max_speed \fInumber\fP] [\-harmonics \fIN\fP] [\-timestep \fInumber\fP] [\-divisor_poisson \fIprobability\fP] [\-size_factor_min \fInumber\fP] [\-size_factor_max \fInumber\fP]
 .SH DESCRIPTION
 The epicycle program draws the path traced out by a point on the edge
diff -ur xscreensaver-4.21.orig/hacks/eruption.man xscreensaver-4.21/hacks/eruption.man
--- xscreensaver-4.21.orig/hacks/eruption.man	2003-01-30 22:21:40.000000000 +0100
+++ xscreensaver-4.21/hacks/eruption.man	2005-05-17 17:19:44.000000000 +0200
@@ -1,8 +1,8 @@
 .TH XScreenSaver 1 "" "X Version 11"
 .SH NAME
-Eruption.
+eruption - eruption of pieces of hot volcanic rock
 .SH SYNOPSIS
-.B Eruption
+.B /usr/lib/xscreensaver/eruption
 [\-display \fIhost:display.screen\fP]
 [\-visual \fIvisual\fP]
 [\-window]
diff -ur xscreensaver-4.21.orig/hacks/euler2d.man xscreensaver-4.21/hacks/euler2d.man
--- xscreensaver-4.21.orig/hacks/euler2d.man	2002-05-31 05:07:52.000000000 +0200
+++ xscreensaver-4.21/hacks/euler2d.man	2005-05-17 17:22:03.000000000 +0200
@@ -1,8 +1,8 @@
 .TH XScreenSaver 1 "" "X Version 11"
 .SH NAME
-euler2d - two dimensional incompressible inviscid fluid flow.
+euler2d - two dimensional incompressible inviscid fluid flow
 .SH SYNOPSIS
-.B euler2d
+.B /usr/lib/xscreensaver/euler2d
 [\-display \fIhost:display.screen\fP]
 [\-visual \fIvisual\fP]
 [\-window]
diff -ur xscreensaver-4.21.orig/hacks/fadeplot.man xscreensaver-4.21/hacks/fadeplot.man
--- xscreensaver-4.21.orig/hacks/fadeplot.man	2002-05-31 04:47:43.000000000 +0200
+++ xscreensaver-4.21/hacks/fadeplot.man	2005-05-17 17:22:06.000000000 +0200
@@ -1,8 +1,8 @@
 .TH XScreenSaver 1 "" "X Version 11"
 .SH NAME
-fadeplot - draws a waving ribbon following a sinusoidal path.
+fadeplot - draws a waving ribbon following a sinusoidal path
 .SH SYNOPSIS
-.B fadeplot
+.B /usr/lib/xscreensaver/fadeplot
 [\-display \fIhost:display.screen\fP]
 [\-visual \fIvisual\fP]
 [\-window]
diff -ur xscreensaver-4.21.orig/hacks/fiberlamp.man xscreensaver-4.21/hacks/fiberlamp.man
--- xscreensaver-4.21.orig/hacks/fiberlamp.man	2005-02-03 03:16:10.000000000 +0100
+++ xscreensaver-4.21/hacks/fiberlamp.man	2005-05-17 17:03:46.000000000 +0200
@@ -2,7 +2,7 @@
 .SH NAME
 fiberlamp - Fiber Optic Lamp
 .SH SYNOPSIS
-.B fiberlamp
+.B /usr/lib/xscreensaver/fiberlamp
 [\-display \fIhost:display.screen\fP]
 [\-visual \fIvisual\fP]
 [\-window]
diff -ur xscreensaver-4.21.orig/hacks/fireworkx.man xscreensaver-4.21/hacks/fireworkx.man
--- xscreensaver-4.21.orig/hacks/fireworkx.man	2004-10-25 17:29:30.000000000 +0200
+++ xscreensaver-4.21/hacks/fireworkx.man	2005-05-17 17:22:14.000000000 +0200
@@ -1,8 +1,8 @@
 .TH XScreenSaver 1 "" "X Version 11"
 .SH NAME
-fireworkx - pyrotechnics simulation eye-candy.
+fireworkx - pyrotechnics simulation eye-candy
 .SH SYNOPSIS
-.B fireworkx
+.B /usr/lib/xscreensaver/fireworkx
 [\-display \fIhost:display.screen\fP]
 [\-visual \fIvisual\fP]
 [\-window]
diff -ur xscreensaver-4.21.orig/hacks/flag.man xscreensaver-4.21/hacks/flag.man
--- xscreensaver-4.21.orig/hacks/flag.man	1998-08-03 10:05:01.000000000 +0200
+++ xscreensaver-4.21/hacks/flag.man	2005-05-17 17:03:57.000000000 +0200
@@ -2,7 +2,7 @@
 .SH NAME
 flag - draws a waving flag, containing text or an image
 .SH SYNOPSIS
-.B flag
+.B /usr/lib/xscreensaver/flag
 [\-display \fIhost:display.screen\fP] [\-foreground \fIcolor\fP] [\-background \fIcolor\fP] [\-window] [\-root] [\-mono] [\-install] [\-visual \fIvisual\fP] [\-ncolors \fIinteger\fP] [\-delay \fImicroseconds\fP] [\-cycles \fIinteger\fP] [\-size \fIinteger\fP] [\-text \fIstring\fP] [\-font \fIfont\fP] [\-bitmap \fIxbm-file\fP]
 
 .SH DESCRIPTION
diff -ur xscreensaver-4.21.orig/hacks/flame.man xscreensaver-4.21/hacks/flame.man
--- xscreensaver-4.21.orig/hacks/flame.man	1998-08-03 10:05:09.000000000 +0200
+++ xscreensaver-4.21/hacks/flame.man	2005-05-17 17:29:20.000000000 +0200
@@ -1,8 +1,8 @@
-.TH XScreenSaver 1 "13-aug-92" "X Version 11"
+.TH XScreenSaver 1 "13-Aug-92" "X Version 11"
 .SH NAME
 flame - draw weird cosmic fractals
 .SH SYNOPSIS
-.B flame
+.B /usr/lib/xscreensaver/flame
 [\-display \fIhost:display.screen\fP] [\-foreground \fIcolor\fP] [\-background \fIcolor\fP] [\-window] [\-root] [\-mono] [\-install] [\-visual \fIvisual\fP] [\-colors \fIinteger\fP] [\-iterations \fIinteger\fP] [\-points \fIinteger\fP] [\-delay \fImicroseconds\fP] [\-delay2 \fImicroseconds\fP]
 .SH DESCRIPTION
 The \fIflame\fP program generates colorful fractal displays.
diff -ur xscreensaver-4.21.orig/hacks/flow.man xscreensaver-4.21/hacks/flow.man
--- xscreensaver-4.21.orig/hacks/flow.man	2004-11-02 21:55:27.000000000 +0100
+++ xscreensaver-4.21/hacks/flow.man	2005-05-17 17:22:19.000000000 +0200
@@ -1,8 +1,8 @@
 .TH XScreenSaver 1 "" "X Version 11"
 .SH NAME
-flow - strange attractors.
+flow - strange attractors
 .SH SYNOPSIS
-.B flow
+.B /usr/lib/xscreensaver/flow
 [\-display \fIhost:display.screen\fP]
 [\-visual \fIvisual\fP]
 [\-window]
diff -ur xscreensaver-4.21.orig/hacks/fluidballs.man xscreensaver-4.21/hacks/fluidballs.man
--- xscreensaver-4.21.orig/hacks/fluidballs.man	2003-02-04 22:48:38.000000000 +0100
+++ xscreensaver-4.21/hacks/fluidballs.man	2005-05-17 17:22:22.000000000 +0200
@@ -1,8 +1,8 @@
 .TH XScreenSaver 1 "" "X Version 11"
 .SH NAME
-fluidballs - the physics of bouncing balls.
+fluidballs - the physics of bouncing balls
 .SH SYNOPSIS
-.B fluidballs
+.B /usr/lib/xscreensaver/fluidballs
 [\-display \fIhost:display.screen\fP]
 [\-visual \fIvisual\fP]
 [\-window]
diff -ur xscreensaver-4.21.orig/hacks/fontglide.man xscreensaver-4.21/hacks/fontglide.man
--- xscreensaver-4.21.orig/hacks/fontglide.man	2005-03-20 21:15:15.000000000 +0100
+++ xscreensaver-4.21/hacks/fontglide.man	2005-05-17 17:04:16.000000000 +0200
@@ -2,7 +2,7 @@
 .SH NAME
 fontglide - characters float onto the screen to form words
 .SH SYNOPSIS
-.B fontglide
+.B /usr/lib/xscreensaver/fontglide
 [\-display \fIhost:display.screen\fP] [\-window] [\-root] [\-install]
 [\-visual \fIvisual\fP] 
 [\-delay \fIusecs\fP] 
diff -ur xscreensaver-4.21.orig/hacks/forest.man xscreensaver-4.21/hacks/forest.man
--- xscreensaver-4.21.orig/hacks/forest.man	2001-09-22 01:57:28.000000000 +0200
+++ xscreensaver-4.21/hacks/forest.man	2005-05-17 17:04:19.000000000 +0200
@@ -2,7 +2,7 @@
 .SH NAME
 forest - draws a fractal forest
 .SH SYNOPSIS
-.B forest
+.B /usr/lib/xscreensaver/forest
 [\-display \fIhost:display.screen\fP] [\-foreground \fIcolor\fP] [\-background \fIcolor\fP] [\-window] [\-root] [\-mono] [\-install] [\-visual \fIvisual\fP] [\-ncolors \fIinteger\fP] [\-delay \fImicroseconds\fP]
 
 .SH DESCRIPTION
diff -ur xscreensaver-4.21.orig/hacks/fuzzyflakes.man xscreensaver-4.21/hacks/fuzzyflakes.man
--- xscreensaver-4.21.orig/hacks/fuzzyflakes.man	2004-07-10 21:44:54.000000000 +0200
+++ xscreensaver-4.21/hacks/fuzzyflakes.man	2005-05-17 17:04:22.000000000 +0200
@@ -2,7 +2,7 @@
 .SH NAME
 fuzzyflakes - falling snowflakes/flower shapes
 .SH SYNOPSIS
-.B fuzzyflakes
+.B /usr/lib/xscreensaver/fuzzyflakes
 [\-display \fIhost:display.screen\fP] [\-foreground \fIcolor\fP]
 [\-background \fIcolor\fP] [\-window] [\-root] [\-mono] [\-install]
 [\-visual \fIvisual\fP] 
diff -ur xscreensaver-4.21.orig/hacks/galaxy.man xscreensaver-4.21/hacks/galaxy.man
--- xscreensaver-4.21.orig/hacks/galaxy.man	2004-12-05 01:14:27.000000000 +0100
+++ xscreensaver-4.21/hacks/galaxy.man	2005-05-17 17:04:37.000000000 +0200
@@ -2,7 +2,7 @@
 .SH NAME
 galaxy - draws spinning galaxies
 .SH SYNOPSIS
-.B galaxy
+.B /usr/lib/xscreensaver/galaxy
 [\-display \fIhost:display.screen\fP] [\-foreground \fIcolor\fP] [\-background \fIcolor\fP] [\-window] [\-root] [\-mono] [\-install] [\-visual \fIvisual\fP] [\-ncolors \fIinteger\fP] [\-delay \fImicroseconds\fP] [\-cycles \fIinteger\fP] [\-count \fIinteger\fP] [\-size \fIinteger\fP] [\-tracks] [\-no\-tracks] [\-spin] [\-no\-spin]
 
 .SH DESCRIPTION
diff -ur xscreensaver-4.21.orig/hacks/goop.man xscreensaver-4.21/hacks/goop.man
--- xscreensaver-4.21.orig/hacks/goop.man	1999-05-30 04:50:24.000000000 +0200
+++ xscreensaver-4.21/hacks/goop.man	2005-05-17 17:04:43.000000000 +0200
@@ -2,7 +2,7 @@
 .SH NAME
 goop - squishy transparent oil and bubble screenhack
 .SH SYNOPSIS
-.B goop
+.B /usr/lib/xscreensaver/goop
 [\-display \fIhost:display.screen\fP] [\-foreground \fIcolor\fP] [\-background \fIcolor\fP] [\-window] [\-root] [\-mono] [\-install] [\-visual \fIvisual\fP] [\-transparent] [\-non\-transparent] [\-additive] [\-subtractive] [\-xor] [\-no\-xor]
 .SH DESCRIPTION
 The \fIgoop\fP program draws a simulation of bubbles in layers of 
diff -ur xscreensaver-4.21.orig/hacks/grav.man xscreensaver-4.21/hacks/grav.man
--- xscreensaver-4.21.orig/hacks/grav.man	2001-02-02 08:05:48.000000000 +0100
+++ xscreensaver-4.21/hacks/grav.man	2005-05-17 17:04:46.000000000 +0200
@@ -2,7 +2,7 @@
 .SH NAME
 grav - draws a simple orbital simulation
 .SH SYNOPSIS
-.B grav
+.B /usr/lib/xscreensaver/grav
 [\-display \fIhost:display.screen\fP] [\-foreground \fIcolor\fP] [\-background \fIcolor\fP] [\-window] [\-root] [\-mono] [\-install] [\-visual \fIvisual\fP] [\-ncolors \fIinteger\fP] [\-delay \fImicroseconds\fP] [\-count \fIinteger\fP] [\-decay] [\-no\-decay] [\-trail] [\-no\-trail]
 
 .SH DESCRIPTION
diff -ur xscreensaver-4.21.orig/hacks/greynetic.man xscreensaver-4.21/hacks/greynetic.man
--- xscreensaver-4.21.orig/hacks/greynetic.man	1998-08-03 10:04:55.000000000 +0200
+++ xscreensaver-4.21/hacks/greynetic.man	2005-05-17 17:29:24.000000000 +0200
@@ -1,8 +1,8 @@
-.TH XScreenSaver 1 "13-aug-92" "X Version 11"
+.TH XScreenSaver 1 "13-Aug-92" "X Version 11"
 .SH NAME
 greynetic - draw random stippled/color rectangles
 .SH SYNOPSIS
-.B greynetic
+.B /usr/lib/xscreensaver/greynetic
 [\-display \fIhost:display.screen\fP] [\-foreground \fIcolor\fP] [\-background \fIcolor\fP] [\-window] [\-root] [\-mono] [\-install] [\-visual \fIvisual\fP] [\-delay \fIusecs\fP]
 .SH DESCRIPTION
 The \fIgreynetic\fP program draws random rectangles.
diff -ur xscreensaver-4.21.orig/hacks/halftone.man xscreensaver-4.21/hacks/halftone.man
--- xscreensaver-4.21.orig/hacks/halftone.man	2002-10-15 10:49:09.000000000 +0200
+++ xscreensaver-4.21/hacks/halftone.man	2005-05-17 17:29:29.000000000 +0200
@@ -1,8 +1,8 @@
-.TH XScreenSaver 1 "9-oct-2002" "X Version 11"
+.TH XScreenSaver 1 "9-Oct-2002" "X Version 11"
 .SH NAME
 halftone - simple halftone pattern of moving mass points
 .SH SYNOPSIS
-.B halftone
+.B /usr/lib/xscreensaver/halftone
 [\-display \fIhost:display.screen\fP]
 [\-visual \fIvisual\fP]
 [\-window]
diff -ur xscreensaver-4.21.orig/hacks/halo.man xscreensaver-4.21/hacks/halo.man
--- xscreensaver-4.21.orig/hacks/halo.man	1998-08-03 10:04:48.000000000 +0200
+++ xscreensaver-4.21/hacks/halo.man	2005-05-17 17:04:56.000000000 +0200
@@ -2,7 +2,7 @@
 .SH NAME
 halo - draw circular patterns
 .SH SYNOPSIS
-.B halo
+.B /usr/lib/xscreensaver/halo
 [\-display \fIhost:display.screen\fP] [\-foreground \fIcolor\fP] [\-background \fIcolor\fP] [\-window] [\-root] [\-mono] [\-install] [\-visual \fIvisual\fP] [\-count \fIint\fP] [\-delay \fIusecs\fP] [\-mode seuss | ramp | random ] [\-animate] [\-colors \fIinteger\fP] [\-cycle] [\-no\-cycle] [\-cycle\-delay \fIusecs\fP] 
 .SH DESCRIPTION
 The \fIhalo\fP program draws cool patterns based on circles.
diff -ur xscreensaver-4.21.orig/hacks/helix.man xscreensaver-4.21/hacks/helix.man
--- xscreensaver-4.21.orig/hacks/helix.man	2002-04-30 13:18:29.000000000 +0200
+++ xscreensaver-4.21/hacks/helix.man	2005-05-17 17:29:33.000000000 +0200
@@ -1,8 +1,8 @@
-.TH XScreenSaver 1 "18-sep-97" "X Version 11"
+.TH XScreenSaver 1 "18-Sep-97" "X Version 11"
 .SH NAME
 helix - draw helical string-art patterns
 .SH SYNOPSIS
-.B helix
+.B /usr/lib/xscreensaver/helix
 [\-display \fIhost:display.screen\fP] [\-foreground \fIcolor\fP] [\-background \fIcolor\fP] [\-window] [\-root] [\-mono] [\-erase\-speed \fIusecs\fP] [\-erase\-mode \fIinteger\fP] [\-delay \fIseconds\fP] [\-install] [\-visual \fIvisual\fP]
 .SH DESCRIPTION
 The \fIhelix\fP program draws interesting patterns composed of line segments
diff -ur xscreensaver-4.21.orig/hacks/hopalong.man xscreensaver-4.21/hacks/hopalong.man
--- xscreensaver-4.21.orig/hacks/hopalong.man	2002-02-16 00:57:59.000000000 +0100
+++ xscreensaver-4.21/hacks/hopalong.man	2005-05-17 17:05:10.000000000 +0200
@@ -2,7 +2,7 @@
 .SH NAME
 hopalong - draw real plane fractals
 .SH SYNOPSIS
-.B hopalong
+.B /usr/lib/xscreensaver/hopalong
 [\-display \fIhost:display.screen\fP] [\-foreground \fIcolor\fP] [\-background \fIcolor\fP] [\-window] [\-root] [\-mono] [\-install] [\-visual \fIvisual\fP] [\-ncolors \fIinteger\fP] [\-delay \fImicroseconds\fP] [\-cycles \fIinteger\fP] [\-count \fIinteger\fP] [\-jong] [\-no\-jong] [\-jong] [\-no\-sine]
 
 .SH DESCRIPTION
diff -ur xscreensaver-4.21.orig/hacks/hyperball.man xscreensaver-4.21/hacks/hyperball.man
--- xscreensaver-4.21.orig/hacks/hyperball.man	2001-02-02 07:40:33.000000000 +0100
+++ xscreensaver-4.21/hacks/hyperball.man	2005-05-17 17:05:13.000000000 +0200
@@ -2,7 +2,7 @@
 .SH NAME
 hyperball - 2d projection of a 4d object
 .SH SYNOPSIS
-.B hyperball
+.B /usr/lib/xscreensaver/hyperball
 [\-display \fIhost:display.screen\fP] [\-foreground \fIcolor\fP] [\-background \fIcolor\fP] [\-xy \fIfloat\fP] [\-xz \fIfloat\fP] [\-yz \fIfloat\fP] [\-xw \fIfloat\fP] [\-yw \fIfloat\fP] [\-zw \fIfloat\fP] [\-observer-z \fIint\fP] [\-delay \fIusecs\fP] [\-window] [\-root] [\-mono] [\-install] [\-visual \fIvisual\fP]
 .SH DESCRIPTION
 The \fIhyperball\fP program displays a wireframe projection of a hyperball
diff -ur xscreensaver-4.21.orig/hacks/hypercube.man xscreensaver-4.21/hacks/hypercube.man
--- xscreensaver-4.21.orig/hacks/hypercube.man	1998-08-03 10:04:57.000000000 +0200
+++ xscreensaver-4.21/hacks/hypercube.man	2005-05-17 17:29:37.000000000 +0200
@@ -1,8 +1,8 @@
-.TH XScreenSaver 1 "6-dec-92" "X Version 11"
+.TH XScreenSaver 1 "6-Dec-92" "X Version 11"
 .SH NAME
 hypercube - 2d projection of a 4d object
 .SH SYNOPSIS
-.B hypercube
+.B /usr/lib/xscreensaver/hypercube
 [\-display \fIhost:display.screen\fP] [\-foreground \fIcolor\fP] [\-background \fIcolor\fP] [\-color[0-7] \fIcolor\fP] [\-xy \fIfloat\fP] [\-xz \fIfloat\fP] [\-yz \fIfloat\fP] [\-xw \fIfloat\fP] [\-yw \fIfloat\fP] [\-zw \fIfloat\fP] [\-observer-z \fIint\fP] [\-delay \fIusecs\fP] [\-window] [\-root] [\-mono] [\-install] [\-visual \fIvisual\fP]
 .SH DESCRIPTION
 The \fIhypercube\fP program displays a wireframe projection of a hypercube
diff -ur xscreensaver-4.21.orig/hacks/ifs.man xscreensaver-4.21/hacks/ifs.man
--- xscreensaver-4.21.orig/hacks/ifs.man	2005-03-18 05:59:50.000000000 +0100
+++ xscreensaver-4.21/hacks/ifs.man	2005-05-17 17:05:20.000000000 +0200
@@ -2,7 +2,7 @@
 .SH NAME
 ifs - draws spinning, colliding iterated-function-system images
 .SH SYNOPSIS
-.B ifs
+.B /usr/lib/xscreensaver/ifs
 [\-display \fIhost:display.screen\fP]
 [\-visual \fIvisual\fP]
 [\-window]
diff -ur xscreensaver-4.21.orig/hacks/imsmap.man xscreensaver-4.21/hacks/imsmap.man
--- xscreensaver-4.21.orig/hacks/imsmap.man	1998-08-03 10:04:54.000000000 +0200
+++ xscreensaver-4.21/hacks/imsmap.man	2005-05-17 17:05:29.000000000 +0200
@@ -2,7 +2,7 @@
 .SH NAME
 imsmap - generate fractal maps
 .SH SYNOPSIS
-.B imsmap
+.B /usr/lib/xscreensaver/imsmap
 [\-display \fIhost:display.screen\fP] [\-foreground \fIcolor\fP] [\-background \fIcolor\fP] [\-window] [\-root] [\-mono] [\-install] [\-visual \fIvisual\fP] [\-ncolors \fIint\fP] [\-delay \fIseconds\fP] [\-iterations \fIint\fP] [\-mode h|s|v|random] [\-cycle] [\-no\-cycle]
 .SH DESCRIPTION
 The \fIimsmap\fP program generates map or cloud-like patterns.  It looks
diff -ur xscreensaver-4.21.orig/hacks/interference.man xscreensaver-4.21/hacks/interference.man
--- xscreensaver-4.21.orig/hacks/interference.man	2003-09-06 08:29:41.000000000 +0200
+++ xscreensaver-4.21/hacks/interference.man	2005-05-17 17:05:33.000000000 +0200
@@ -2,7 +2,7 @@
 .SH NAME
 interference - decaying sinusoidal waves
 .SH SYNOPSIS
-.B interference
+.B /usr/lib/xscreensaver/interference
 [\-display \fIhost:display.screen\fP]
 [\-visual \fIvisual\fP]
 [\-window]
diff -ur xscreensaver-4.21.orig/hacks/intermomentary.man xscreensaver-4.21/hacks/intermomentary.man
--- xscreensaver-4.21.orig/hacks/intermomentary.man	2004-10-24 02:20:37.000000000 +0200
+++ xscreensaver-4.21/hacks/intermomentary.man	2005-05-17 17:23:04.000000000 +0200
@@ -1,9 +1,9 @@
-.TH Intermomentary 1 "08-Oct-04" "X Version 11"
+.TH XScreenSaver 1 "08-Oct-04" "X Version 11"
 .SH NAME
-intermomentary - Visualize the momentary and aggregate intersections 
+intermomentary - visualize the momentary and aggregate intersections 
 between slowly moving circles.
 .SH SYNOPSIS
-.B intermomentary
+.B /usr/lib/xscreensaver/intermomentary
 [\-display \fIhost:display.screen\fP]
 [\-foreground \fIcolor\fP]
 [\-background \fIcolor\fP]
diff -ur xscreensaver-4.21.orig/hacks/jigsaw.man xscreensaver-4.21/hacks/jigsaw.man
--- xscreensaver-4.21.orig/hacks/jigsaw.man	2001-04-15 10:55:40.000000000 +0200
+++ xscreensaver-4.21/hacks/jigsaw.man	2005-05-17 17:05:40.000000000 +0200
@@ -2,7 +2,7 @@
 .SH NAME
 jigsaw - permute the screen image like a jigsaw puzzle
 .SH SYNOPSIS
-.B jigsaw
+.B /usr/lib/xscreensaver/jigsaw
 [\-display \fIhost:display.screen\fP] [\-background \fIcolor\fP]
 [\-delay \fIusecs\fP] [\-window] [\-root] [\-install] [\-visual \fIvisual\fP]
 .SH DESCRIPTION
diff -ur xscreensaver-4.21.orig/hacks/juggle.man xscreensaver-4.21/hacks/juggle.man
--- xscreensaver-4.21.orig/hacks/juggle.man	2005-02-03 03:15:36.000000000 +0100
+++ xscreensaver-4.21/hacks/juggle.man	2005-05-17 17:23:13.000000000 +0200
@@ -2,9 +2,9 @@
 .\" ** The above line should force tbl to be used as a preprocessor **
 .TH XScreenSaver 1 "" "X Version 11"
 .SH NAME
-juggle \- juggling man screen saver.
+juggle \- juggling man screen saver
 .SH SYNOPSIS
-.B juggle
+.B /usr/lib/xscreensaver/juggle
 .RB [ \-display
 .IR host:display.screen ]
 .RB [ \-root ]
diff -ur xscreensaver-4.21.orig/hacks/julia.man xscreensaver-4.21/hacks/julia.man
--- xscreensaver-4.21.orig/hacks/julia.man	1998-08-03 10:05:00.000000000 +0200
+++ xscreensaver-4.21/hacks/julia.man	2005-05-17 17:05:56.000000000 +0200
@@ -2,7 +2,7 @@
 .SH NAME
 julia - draws spinning, animating julia-set fractals
 .SH SYNOPSIS
-.B julia
+.B /usr/lib/xscreensaver/julia
 [\-display \fIhost:display.screen\fP] [\-foreground \fIcolor\fP] [\-background \fIcolor\fP] [\-window] [\-root] [\-mono] [\-install] [\-visual \fIvisual\fP] [\-ncolors \fIinteger\fP] [\-delay \fImicroseconds\fP] [\-cycles \fIinteger\fP] [\-count \fIinteger\fP] [\-mouse] [\-nomouse]
 
 .SH DESCRIPTION
diff -ur xscreensaver-4.21.orig/hacks/kaleidescope.man xscreensaver-4.21/hacks/kaleidescope.man
--- xscreensaver-4.21.orig/hacks/kaleidescope.man	1997-04-28 02:47:22.000000000 +0200
+++ xscreensaver-4.21/hacks/kaleidescope.man	2005-05-17 17:23:28.000000000 +0200
@@ -11,11 +11,11 @@
 .if n .sp 1
 .if t .sp .5
 ..
-.TH Kaleidescpe 1 "14-Dec-95" "X Version 11"
+.TH XScreenSaver 1 "14-Dec-95" "X Version 11"
 .SH NAME
-Kaleidescope - rotating line segments
+kaleidescope - rotating line segments
 .SH SYNOPSIS
-.B kaleidescope
+.B /usr/lib/xscreensaver/kaleidescope
 [\-display \fIhost:display.screen\fP] [\-foreground \fIcolor\fP] [\-background \fIcolor\fP] [\-window] [\-root] [\-install] [\-visual \fIvisual\fP] [\-color_mode \fImono | nice | greedy\fP] [-nsegments \fIint\fP] [\-ntrails \fIint\fP] [\-local_rotation \fIint\fP] [\-global_rotation \fIint\fP] [\-delay \fIusecs\fP] [\-redmin \fIint\fP] [\-greenmin \fIint\fP] [\-bluemin \fIint\fP] [\-redrange \fIint\fP] [\-greenrange \fIint\fP] [\-bluerange \fIint\fP]
 .SH DESCRIPTION
 The \fIkaleidescope\fP program draws line segments in a symmetric pattern
diff -ur xscreensaver-4.21.orig/hacks/kumppa.man xscreensaver-4.21/hacks/kumppa.man
--- xscreensaver-4.21.orig/hacks/kumppa.man	2002-05-31 05:00:55.000000000 +0200
+++ xscreensaver-4.21/hacks/kumppa.man	2005-05-17 17:23:31.000000000 +0200
@@ -1,8 +1,8 @@
 .TH XScreenSaver 1 "" "X Version 11"
 .SH NAME
-kumppa - spiraling, spinning, splashes of color rush toward the screen.
+kumppa - spiraling, spinning, splashes of color rush toward the screen
 .SH SYNOPSIS
-.B kumppa
+.B /usr/lib/xscreensaver/kumppa
 [\-display \fIhost:display.screen\fP]
 [\-visual \fIvisual\fP]
 [\-window]
diff -ur xscreensaver-4.21.orig/hacks/laser.man xscreensaver-4.21/hacks/laser.man
--- xscreensaver-4.21.orig/hacks/laser.man	1998-08-03 10:05:04.000000000 +0200
+++ xscreensaver-4.21/hacks/laser.man	2005-05-17 17:06:55.000000000 +0200
@@ -2,7 +2,7 @@
 .SH NAME
 laser - draws vaguely laser-like moving lines
 .SH SYNOPSIS
-.B laser
+.B /usr/lib/xscreensaver/laser
 [\-display \fIhost:display.screen\fP] [\-foreground \fIcolor\fP] [\-background \fIcolor\fP] [\-window] [\-root] [\-mono] [\-install] [\-visual \fIvisual\fP] [\-ncolors \fIinteger\fP] [\-delay \fImicroseconds\fP] [\-cycles \fIinteger\fP] [\-count \fIinteger\fP]
 
 .SH DESCRIPTION
diff -ur xscreensaver-4.21.orig/hacks/lightning.man xscreensaver-4.21/hacks/lightning.man
--- xscreensaver-4.21.orig/hacks/lightning.man	1998-08-03 10:05:15.000000000 +0200
+++ xscreensaver-4.21/hacks/lightning.man	2005-05-17 17:07:00.000000000 +0200
@@ -2,7 +2,7 @@
 .SH NAME
 lightning - draws fractal lightning bolts
 .SH SYNOPSIS
-.B lightning
+.B /usr/lib/xscreensaver/lightning
 [\-display \fIhost:display.screen\fP] [\-foreground \fIcolor\fP] [\-background \fIcolor\fP] [\-window] [\-root] [\-mono] [\-install] [\-visual \fIvisual\fP] [\-ncolors \fIinteger\fP] [\-delay \fImicroseconds\fP]
 
 .SH DESCRIPTION
diff -ur xscreensaver-4.21.orig/hacks/lisa.man xscreensaver-4.21/hacks/lisa.man
--- xscreensaver-4.21.orig/hacks/lisa.man	1998-08-03 10:05:14.000000000 +0200
+++ xscreensaver-4.21/hacks/lisa.man	2005-05-17 17:07:13.000000000 +0200
@@ -2,7 +2,7 @@
 .SH NAME
 lisa - draws animated full-loop lisajous figures
 .SH SYNOPSIS
-.B lisa
+.B /usr/lib/xscreensaver/lisa
 [\-display \fIhost:display.screen\fP] [\-foreground \fIcolor\fP] [\-background \fIcolor\fP] [\-window] [\-root] [\-mono] [\-install] [\-visual \fIvisual\fP] [\-ncolors \fIinteger\fP] [\-delay \fImicroseconds\fP] [\-cycles \fIinteger\fP] [\-count \fIinteger\fP] [\-size \fIinteger\fP]
 
 .SH DESCRIPTION
diff -ur xscreensaver-4.21.orig/hacks/lissie.man xscreensaver-4.21/hacks/lissie.man
--- xscreensaver-4.21.orig/hacks/lissie.man	2002-05-31 04:51:17.000000000 +0200
+++ xscreensaver-4.21/hacks/lissie.man	2005-05-17 17:23:39.000000000 +0200
@@ -1,8 +1,8 @@
 .TH XScreenSaver 1 "" "X Version 11"
 .SH NAME
-lissie - lissajous figure.
+lissie - lissajous figure
 .SH SYNOPSIS
-.B lissie
+.B /usr/lib/xscreensaver/lissie
 [\-display \fIhost:display.screen\fP]
 [\-visual \fIvisual\fP]
 [\-window]
diff -ur xscreensaver-4.21.orig/hacks/ljlatest.man xscreensaver-4.21/hacks/ljlatest.man
--- xscreensaver-4.21.orig/hacks/ljlatest.man	2005-03-20 21:21:00.000000000 +0100
+++ xscreensaver-4.21/hacks/ljlatest.man	2005-05-17 17:23:44.000000000 +0200
@@ -13,9 +13,9 @@
 ..
 .TH XScreenSaver 1 "20-Mar-2005" "X Version 11"
 .SH NAME
-ljlatest - prints the most recent public posts on livejournal.com.
+ljlatest - prints the most recent public posts on livejournal.com
 .SH SYNOPSIS
-.B ljlatest
+.B /usr/lib/xscreensaver/ljlatest
 [\--verbose]
 [\--columns \fIN\fP]
 .SH DESCRIPTION
diff -ur xscreensaver-4.21.orig/hacks/lmorph.man xscreensaver-4.21/hacks/lmorph.man
--- xscreensaver-4.21.orig/hacks/lmorph.man	2001-12-20 09:20:59.000000000 +0100
+++ xscreensaver-4.21/hacks/lmorph.man	2005-05-17 17:24:02.000000000 +0200
@@ -1,8 +1,8 @@
-.TH LMORPH 1 "xscreensaver hack"
+.TH XScreenSaver 1
 .SH NAME
 lmorph \- morphing lines
 .SH SYNOPSIS
-.B lmorph
+.B /usr/lib/xscreensaver/lmorph
 [\-display \fIhost:display.screen\fP] [\-foreground \fIcolor\fP] [\-background \fIcolor\fP] [\-window] [\-root] [\-mono] [\-install] [\-visual \fIvisual\fP] [\-points \fIint\fP] [\-steps \fIint\fP] [\-delay \fIusecs\fP] [\-figtype \fItype\fP]
 .SH DESCRIPTION
 The \fIlmorph\fP program shows smooth and non-linear morphing between
diff -ur xscreensaver-4.21.orig/hacks/loop.man xscreensaver-4.21/hacks/loop.man
--- xscreensaver-4.21.orig/hacks/loop.man	2002-05-31 05:01:59.000000000 +0200
+++ xscreensaver-4.21/hacks/loop.man	2005-05-17 17:24:05.000000000 +0200
@@ -1,8 +1,8 @@
 .TH XScreenSaver 1 "" "X Version 11"
 .SH NAME
-loop - cellular automaton.
+loop - cellular automaton
 .SH SYNOPSIS
-.B loop
+.B /usr/lib/xscreensaver/loop
 [\-display \fIhost:display.screen\fP]
 [\-visual \fIvisual\fP]
 [\-window]
diff -ur xscreensaver-4.21.orig/hacks/maze.man xscreensaver-4.21/hacks/maze.man
--- xscreensaver-4.21.orig/hacks/maze.man	1998-06-29 04:36:18.000000000 +0200
+++ xscreensaver-4.21/hacks/maze.man	2005-05-17 17:29:40.000000000 +0200
@@ -1,8 +1,8 @@
-.TH XScreenSaver 1 "7-mar-93" "X Version 11"
+.TH XScreenSaver 1 "7-Mar-93" "X Version 11"
 .SH NAME
 maze \- an automated X11 demo repeatedly creating and solving a random maze
 .SH SYNOPSIS
-.B maze 
+.B /usr/lib/xscreensaver/maze 
 [\-display \fIhost:display.screen\fP] [\-foreground \fIcolor\fP] [\-background \fIcolor\fP] [\-window] [\-root] [\-install] [\-visual \fIvisual\fP] [\-grid\-size \fIpixels\fP] [\-live\-color \fIcolor\fP] [\-dead\-color \fIcolor\fP] [\-solve\-delay \fIusecs\fP] [\-pre\-delay \fIusecs\fP] [\-post\-delay \fIusecs\fP] [\-generator \fIinteger\fP] [\-max\-length \fIinteger\fP] [\-bridge] [\-no\-bridge]
 .SH DESCRIPTION
 The \fImaze\fP program creates a "random" maze and then solves it with 
diff -ur xscreensaver-4.21.orig/hacks/memscroller.man xscreensaver-4.21/hacks/memscroller.man
--- xscreensaver-4.21.orig/hacks/memscroller.man	2004-08-18 06:05:48.000000000 +0200
+++ xscreensaver-4.21/hacks/memscroller.man	2005-05-17 17:07:56.000000000 +0200
@@ -2,7 +2,7 @@
 .SH NAME
 memscroller - scrolls a dump of its own RAM across the screen
 .SH SYNOPSIS
-.B memscroller
+.B /usr/lib/xscreensaver/memscroller
 [\-display \fIhost:display.screen\fP]
 [\-window] [\-root] [\-mono] [\-install] [\-visual \fIvisual\fP]
 [\-font \fIfont\fP]
diff -ur xscreensaver-4.21.orig/hacks/metaballs.man xscreensaver-4.21/hacks/metaballs.man
--- xscreensaver-4.21.orig/hacks/metaballs.man	2003-01-25 10:17:34.000000000 +0100
+++ xscreensaver-4.21/hacks/metaballs.man	2005-05-17 17:19:59.000000000 +0200
@@ -1,8 +1,8 @@
 .TH XScreenSaver 1 "" "X Version 11"
 .SH NAME
-MetaBalls.
+metaballs - draws 2D metaballs
 .SH SYNOPSIS
-.B MetaBalls
+.B /usr/lib/xscreensaver/metaballs
 [\-display \fIhost:display.screen\fP]
 [\-visual \fIvisual\fP]
 [\-window]
diff -ur xscreensaver-4.21.orig/hacks/mismunch.man xscreensaver-4.21/hacks/mismunch.man
--- xscreensaver-4.21.orig/hacks/mismunch.man	2004-02-12 02:14:42.000000000 +0100
+++ xscreensaver-4.21/hacks/mismunch.man	2005-05-17 17:08:16.000000000 +0200
@@ -2,7 +2,7 @@
 .SH NAME
 mismunch - munches errors
 .SH SYNOPSIS
-.B mismunch
+.B /usr/lib/xscreensaver/mismunch
 [\-display \fIhost:display.screen\fP] [\-foreground \fIcolor\fP]
 [\-background \fIcolor\fP] [\-window] [\-root] [\-mono] [\-install]
 [\-visual \fIvisual\fP] [\-delay \fIusecs\fP] [\-xor] [\-noxor] 
diff -ur xscreensaver-4.21.orig/hacks/moire.man xscreensaver-4.21/hacks/moire.man
--- xscreensaver-4.21.orig/hacks/moire.man	2003-06-19 12:48:21.000000000 +0200
+++ xscreensaver-4.21/hacks/moire.man	2005-05-17 17:08:20.000000000 +0200
@@ -2,7 +2,7 @@
 .SH NAME
 moire - draw circular interference patterns
 .SH SYNOPSIS
-.B moire
+.B /usr/lib/xscreensaver/moire
 [\-display \fIhost:display.screen\fP] [\-foreground \fIcolor\fP] [\-background \fIcolor\fP] [\-window] [\-root] [\-mono] [\-install] [\-visual \fIvisual\fP] [\-delay \fIseconds\fP] [\-random \fIboolean\fP] [\-ncolors \fIint\fP] [\-offset \fIint\fP] 
 .SH DESCRIPTION
 The \fImoire\fP program draws cool circular interference patterns.
diff -ur xscreensaver-4.21.orig/hacks/moire2.man xscreensaver-4.21/hacks/moire2.man
--- xscreensaver-4.21.orig/hacks/moire2.man	2002-05-31 04:55:54.000000000 +0200
+++ xscreensaver-4.21/hacks/moire2.man	2005-05-17 17:24:17.000000000 +0200
@@ -1,8 +1,8 @@
 .TH XScreenSaver 1 "" "X Version 11"
 .SH NAME
-moire2 - circular interference patterns.
+moire2 - circular interference patterns
 .SH SYNOPSIS
-.B moire2
+.B /usr/lib/xscreensaver/moire2
 [\-display \fIhost:display.screen\fP]
 [\-visual \fIvisual\fP]
 [\-window]
diff -ur xscreensaver-4.21.orig/hacks/mountain.man xscreensaver-4.21/hacks/mountain.man
--- xscreensaver-4.21.orig/hacks/mountain.man	2002-05-31 04:49:17.000000000 +0200
+++ xscreensaver-4.21/hacks/mountain.man	2005-05-17 17:24:20.000000000 +0200
@@ -1,8 +1,8 @@
 .TH XScreenSaver 1 "" "X Version 11"
 .SH NAME
-mountain - random 3D plots that look vaguely mountainous.
+mountain - random 3D plots that look vaguely mountainous
 .SH SYNOPSIS
-.B mountain
+.B /usr/lib/xscreensaver/mountain
 [\-display \fIhost:display.screen\fP]
 [\-visual \fIvisual\fP]
 [\-window]
diff -ur xscreensaver-4.21.orig/hacks/munch.man xscreensaver-4.21/hacks/munch.man
--- xscreensaver-4.21.orig/hacks/munch.man	1998-11-15 20:37:33.000000000 +0100
+++ xscreensaver-4.21/hacks/munch.man	2005-05-17 17:08:40.000000000 +0200
@@ -15,7 +15,7 @@
 .SH NAME
 munch - munching squares screen hack
 .SH SYNOPSIS
-.B munch
+.B /usr/lib/xscreensaver/munch
 [\-display \fIhost:display.screen\fP] [\-foreground \fIcolor\fP]
 [\-background \fIcolor\fP] [\-window] [\-root] [\-mono] [\-install]
 [\-visual \fIvisual\fP] [\-delay \fIseconds\fP] [\-xor] [\-noxor] [\-shift]
diff -ur xscreensaver-4.21.orig/hacks/nerverot.man xscreensaver-4.21/hacks/nerverot.man
--- xscreensaver-4.21.orig/hacks/nerverot.man	2005-02-24 07:24:08.000000000 +0100
+++ xscreensaver-4.21/hacks/nerverot.man	2005-05-17 17:08:44.000000000 +0200
@@ -2,7 +2,7 @@
 .SH NAME
 nerverot - induces edginess in the viewer
 .SH SYNOPSIS
-.B nerverot
+.B /usr/lib/xscreensaver/nerverot
 [\-display \fIhost:display.screen\fP] [\-foreground \fIcolor\fP] [\-background \fIcolor\fP] [\-window] [\-root] [\-mono] [\-install] [\-visual \fIvisual\fP] [\-db] [\-no\-db] [\-colors \fIinteger\fP] [\-delay \fImicroseconds\fP] [\-count \fIinteger\fP] [\-line\-width \fIinteger\fP] [\-event\-chance \fIfraction\fP] [\-iter\-amt \fIfraction\fP] [\-nervousness \fIfraction\fP] [\-max\-nerve\-radius \fIfraction\fP] [\-min\-radius \fIinteger\fP] [\-max\-radius \fIinteger\fP] [\-min\-scale \fIfraction\fP] [\-max\-scale \fIfraction\fP]
 .SH DESCRIPTION
 The goal of \fInerverot\fP is to be interesting and compelling to
diff -ur xscreensaver-4.21.orig/hacks/noseguy.man xscreensaver-4.21/hacks/noseguy.man
--- xscreensaver-4.21.orig/hacks/noseguy.man	2005-03-20 23:16:18.000000000 +0100
+++ xscreensaver-4.21/hacks/noseguy.man	2005-05-17 17:30:12.000000000 +0200
@@ -1,8 +1,8 @@
-.TH XScreenSaver 1 "13-aug-92" "X Version 11"
+.TH XScreenSaver 1 "13-Aug-92" "X Version 11"
 .SH NAME
 noseguy - a little guy with a big nose wanders around being witty
 .SH SYNOPSIS
-.B noseguy
+.B /usr/lib/xscreensaver/noseguy
 [\-display \fIhost:display.screen\fP]
 [\-foreground \fIcolor\fP]
 [\-background \fIcolor\fP]
diff -ur xscreensaver-4.21.orig/hacks/pacman.man xscreensaver-4.21/hacks/pacman.man
--- xscreensaver-4.21.orig/hacks/pacman.man	2004-02-13 08:32:11.000000000 +0100
+++ xscreensaver-4.21/hacks/pacman.man	2005-05-17 17:24:27.000000000 +0200
@@ -1,8 +1,8 @@
 .TH XScreenSaver 1 "12-Feb-2004" "X Version 11"
 .SH NAME
-pacman - simulates a game of Pac-Man on a randomly-created level.
+pacman - simulates a game of Pac-Man on a randomly-created level
 .SH SYNOPSIS
-.B pacman
+.B /usr/lib/xscreensaver/pacman
 [\-display \fIhost:display.screen\fP]
 [\-window]
 [\-root]
diff -ur xscreensaver-4.21.orig/hacks/pedal.man xscreensaver-4.21/hacks/pedal.man
--- xscreensaver-4.21.orig/hacks/pedal.man	1995-01-11 03:37:05.000000000 +0100
+++ xscreensaver-4.21/hacks/pedal.man	2005-05-17 17:09:00.000000000 +0200
@@ -2,7 +2,7 @@
 .SH NAME
 pedal - pretty geometric picture program
 .SH SYNOPSIS
-.B pedal
+.B /usr/lib/xscreensaver/pedal
 [\-display \fIhost:display.screen\fP] [\-foreground \fIcolor\fP] [\-background \fIcolor\fP] [\-window] [\-root] [\-delay \fIseconds\fP] [-maxlines \fInumber\fP] [-fadedelay \fIuseconds\fP] [-mono] [\-install] [\-visual \fIvisual\fP]
 .SH DESCRIPTION
 The \fIpedal\fP program displays pretty geometric pictures.
diff -ur xscreensaver-4.21.orig/hacks/penetrate.man xscreensaver-4.21/hacks/penetrate.man
--- xscreensaver-4.21.orig/hacks/penetrate.man	2001-09-22 01:48:07.000000000 +0200
+++ xscreensaver-4.21/hacks/penetrate.man	2005-05-17 17:09:04.000000000 +0200
@@ -15,7 +15,7 @@
 .SH NAME
 penetrate - simulates a classic arcade shooting game
 .SH SYNOPSIS
-.B penetrate
+.B /usr/lib/xscreensaver/penetrate
 [\-display \fIhost:display.screen\fP] [\-root] [\-window]
 [\-install] [\-noinstall] [\-visual \fIvisual\fP]
 [\-bgrowth \fImicroseconds\fP] [\-lrate \fInumber\fP] [\-smart \fInumber\fP]
diff -ur xscreensaver-4.21.orig/hacks/penrose.man xscreensaver-4.21/hacks/penrose.man
--- xscreensaver-4.21.orig/hacks/penrose.man	2002-06-15 08:21:01.000000000 +0200
+++ xscreensaver-4.21/hacks/penrose.man	2005-05-17 17:09:07.000000000 +0200
@@ -2,7 +2,7 @@
 .SH NAME
 penrose - draws quasiperiodic tilings
 .SH SYNOPSIS
-.B penrose
+.B /usr/lib/xscreensaver/penrose
 [\-display \fIhost:display.screen\fP] [\-foreground \fIcolor\fP] [\-background \fIcolor\fP] [\-window] [\-root] [\-mono] [\-install] [\-visual \fIvisual\fP] [\-ncolors \fIinteger\fP] [\-delay \fImicroseconds\fP] [\-redoDelay \fIseconds\fP] [\-size \fIinteger\fP] [\-ammann] [\-no\-ammann]
 
 .SH DESCRIPTION
diff -ur xscreensaver-4.21.orig/hacks/petri.man xscreensaver-4.21/hacks/petri.man
--- xscreensaver-4.21.orig/hacks/petri.man	2000-07-18 07:59:15.000000000 +0200
+++ xscreensaver-4.21/hacks/petri.man	2005-05-17 17:09:11.000000000 +0200
@@ -2,7 +2,7 @@
 .SH NAME
 petri - simulates mold growing in a petri dish
 .SH SYNOPSIS
-.B petri
+.B /usr/lib/xscreensaver/petri
 [\-display \fIhost:display.screen\fP] [\-foreground \fIcolor\fP] [\-background \fIcolor\fP] [\-window] [\-root] [\-mono] [\-install] [\-visual \fIvisual\fP] [\-delay \fImicroseconds\fP] [\-size \fIinteger\fP] [\-mem-throttle \fIamount\fP] [\-count \fIinteger\fP] [\-originalcolors] [\-diaglim \fIreal\fP] [\-anychan \fIreal\fP] [\-minorchan \fIreal\fP] [\-instantdeathchan \fIreal\fP] [\-minlifespeed \fIreal\fP] [\-maxlifespeed \fIreal\fP] [\-mindeathspeed \fIreal\fP] [\-maxdeathspeed \fIreal\fP] [\-minlifespan \fIinteger\fP] [\-maxlifespan \fIinteger\fP]
 .SH DESCRIPTION
 \fIpetri\fP simulates mold growing in a petri dish via a state-heavy grid
diff -ur xscreensaver-4.21.orig/hacks/phosphor.man xscreensaver-4.21/hacks/phosphor.man
--- xscreensaver-4.21.orig/hacks/phosphor.man	2005-03-20 22:53:18.000000000 +0100
+++ xscreensaver-4.21/hacks/phosphor.man	2005-05-17 17:09:28.000000000 +0200
@@ -15,7 +15,7 @@
 .SH NAME
 phosphor - simulates an old terminal with long-sustain phosphor
 .SH SYNOPSIS
-.B phosphor
+.B /usr/lib/xscreensaver/phosphor
 [\-display \fIhost:display.screen\fP] [\-window] [\-root] [\-install]
 [\-visual \fIvisual\fP] [\-font \fIfont\fP] [\-scale \fIint\fP]
 [\-ticks \fIint\fP] [\-delay \fIusecs\fP] [\-program \fIcommand\fP]
diff -ur xscreensaver-4.21.orig/hacks/piecewise.man xscreensaver-4.21/hacks/piecewise.man
--- xscreensaver-4.21.orig/hacks/piecewise.man	2003-02-18 22:15:16.000000000 +0100
+++ xscreensaver-4.21/hacks/piecewise.man	2005-05-17 17:24:38.000000000 +0200
@@ -1,8 +1,8 @@
 .TH XScreenSaver 1 "" "X Version 11"
 .SH NAME
-piecewise - lots of moving circles intersecting in interesting ways.
+piecewise - lots of moving circles intersecting in interesting ways
 .SH SYNOPSIS
-.B piecewise
+.B /usr/lib/xscreensaver/piecewise
 [\-display \fIhost:display.screen\fP]
 [\-visual \fIvisual\fP]
 [\-window]
diff -ur xscreensaver-4.21.orig/hacks/polyominoes.man xscreensaver-4.21/hacks/polyominoes.man
--- xscreensaver-4.21.orig/hacks/polyominoes.man	2002-05-31 05:08:44.000000000 +0200
+++ xscreensaver-4.21/hacks/polyominoes.man	2005-05-17 17:24:42.000000000 +0200
@@ -1,8 +1,8 @@
 .TH XScreenSaver 1 "" "X Version 11"
 .SH NAME
-polyominoes - fill a rectangle with irregularly-shaped blocks.
+polyominoes - fill a rectangle with irregularly-shaped blocks
 .SH SYNOPSIS
-.B polyominoes
+.B /usr/lib/xscreensaver/polyominoes
 [\-display \fIhost:display.screen\fP]
 [\-visual \fIvisual\fP]
 [\-window]
diff -ur xscreensaver-4.21.orig/hacks/pong.man xscreensaver-4.21/hacks/pong.man
--- xscreensaver-4.21.orig/hacks/pong.man	2004-05-06 06:26:49.000000000 +0200
+++ xscreensaver-4.21/hacks/pong.man	2005-05-17 17:09:38.000000000 +0200
@@ -2,7 +2,7 @@
 .SH NAME
 pong - Pong Home Video Game Emulator
 .SH SYNOPSIS
-.B pong
+.B /usr/lib/xscreensaver/pong
 [\-display \fIhost:display.screen\fP] [\-foreground \fIcolor\fP]
 [\-background \fIcolor\fP] [\-window] [\-root] [\-mono] [\-install]
 [\-visual \fIvisual\fP]
diff -ur xscreensaver-4.21.orig/hacks/pyro.man xscreensaver-4.21/hacks/pyro.man
--- xscreensaver-4.21.orig/hacks/pyro.man	1998-08-03 10:04:47.000000000 +0200
+++ xscreensaver-4.21/hacks/pyro.man	2005-05-17 17:30:16.000000000 +0200
@@ -1,8 +1,8 @@
-.TH XScreenSaver 1 "13-aug-92" "X Version 11"
+.TH XScreenSaver 1 "13-Aug-92" "X Version 11"
 .SH NAME
 pyro - simulate fireworks
 .SH SYNOPSIS
-.B pyro
+.B /usr/lib/xscreensaver/pyro
 [\-display \fIhost:display.screen\fP] [\-foreground \fIcolor\fP] [\-background \fIcolor\fP] [\-window] [\-root] [\-mono] [\-install] [\-visual \fIvisual\fP] [\-count \fIinteger\fP] [\-frequency \fIinteger\fP] [\-scatter \fIinteger\fP]
 .SH DESCRIPTION
 The \fIpyro\fP program simulates fireworks, in a way similar to a Macintosh
diff -ur xscreensaver-4.21.orig/hacks/qix.man xscreensaver-4.21/hacks/qix.man
--- xscreensaver-4.21.orig/hacks/qix.man	1999-05-30 04:50:09.000000000 +0200
+++ xscreensaver-4.21/hacks/qix.man	2005-05-17 17:09:51.000000000 +0200
@@ -2,7 +2,7 @@
 .SH NAME
 qix - bounce colored lines around a window
 .SH SYNOPSIS
-.B qix
+.B /usr/lib/xscreensaver/qix
 [\-display \fIhost:display.screen\fP] [\-foreground \fIcolor\fP] [\-background \fIcolor\fP] [\-window] [\-root] [\-mono] [\-install] [\-visual \fIvisual\fP] [\-segments \fIint\fP] [\-spread \fIpixels\fP] [\-size \fIpixels\fP] [\-count \fIint\fP] [\-color-shift \fIdegrees\fP] [\-delay \fIusecs\fP] [\-random] [\-linear] [\-solid] [\-hollow] [\-xor] [\-no\-xor] [\-transparent] [\-non\-transparent] [\-additive] [\-subtractive] [\-poly \fIint\fP] [\-gravity] [\-no\-gravity]
 .SH DESCRIPTION
 The \fIqix\fP program bounces a series of line segments around its window.
diff -ur xscreensaver-4.21.orig/hacks/rd-bomb.man xscreensaver-4.21/hacks/rd-bomb.man
--- xscreensaver-4.21.orig/hacks/rd-bomb.man	1997-10-07 08:28:07.000000000 +0200
+++ xscreensaver-4.21/hacks/rd-bomb.man	2005-05-17 17:09:54.000000000 +0200
@@ -2,7 +2,7 @@
 .SH NAME
 rd-bomb - reaction/diffusion textures
 .SH SYNOPSIS
-.B rd-bomb
+.B /usr/lib/xscreensaver/rd-bomb
 [\-display \fIhost:display.screen\fP] [\-foreground \fIcolor\fP]
 [\-background \fIcolor\fP] [\-window] [\-root] [\-install]
 [\-visual \fIvisual\fP] [\-width \fIn\fP] [\-height \fIn\fP]
diff -ur xscreensaver-4.21.orig/hacks/ripples.man xscreensaver-4.21/hacks/ripples.man
--- xscreensaver-4.21.orig/hacks/ripples.man	2002-05-31 05:05:47.000000000 +0200
+++ xscreensaver-4.21/hacks/ripples.man	2005-05-17 17:24:52.000000000 +0200
@@ -1,8 +1,8 @@
 .TH XScreenSaver 1 "" "X Version 11"
 .SH NAME
-ripples - interference patterns.
+ripples - interference patterns
 .SH SYNOPSIS
-.B ripples
+.B /usr/lib/xscreensaver/ripples
 [\-display \fIhost:display.screen\fP]
 [\-visual \fIvisual\fP]
 [\-window]
diff -ur xscreensaver-4.21.orig/hacks/rocks.man xscreensaver-4.21/hacks/rocks.man
--- xscreensaver-4.21.orig/hacks/rocks.man	1998-08-03 10:05:03.000000000 +0200
+++ xscreensaver-4.21/hacks/rocks.man	2005-05-17 17:30:20.000000000 +0200
@@ -1,8 +1,8 @@
-.TH XScreenSaver 1 "13-aug-92" "X Version 11"
+.TH XScreenSaver 1 "13-Aug-92" "X Version 11"
 .SH NAME
 rocks - animation of flying through an asteroid field
 .SH SYNOPSIS
-.B rocks
+.B /usr/lib/xscreensaver/rocks
 [\-display \fIhost:display.screen\fP] [\-foreground \fIcolor\fP] [\-background \fIcolor\fP] [\-window] [\-root] [\-mono] [\-ncolors \fIn\fP] [\-install] [\-visual \fIvisual\fP] [\-count \fIinteger\fP] [\-delay \fIusecs\fP] [\-speed \fIinteger\fP] [\-norotate] [\-nomove] [\-3d]
 .SH DESCRIPTION
 The \fIrocks\fP program draws an animation of an asteroid field moving past
diff -ur xscreensaver-4.21.orig/hacks/rorschach.man xscreensaver-4.21/hacks/rorschach.man
--- xscreensaver-4.21.orig/hacks/rorschach.man	1998-08-03 10:05:02.000000000 +0200
+++ xscreensaver-4.21/hacks/rorschach.man	2005-05-17 17:30:23.000000000 +0200
@@ -1,8 +1,8 @@
-.TH XScreenSaver 1 "13-aug-92" "X Version 11"
+.TH XScreenSaver 1 "13-Aug-92" "X Version 11"
 .SH NAME
 rorschach - simulate ink-blot patterns
 .SH SYNOPSIS
-.B rorschach
+.B /usr/lib/xscreensaver/rorschach
 [\-display \fIhost:display.screen\fP] [\-foreground \fIcolor\fP] [\-background \fIcolor\fP] [\-window] [\-root] [\-mono] [\-install] [\-visual \fIvisual\fP] [\-iterations \fIinteger\fP] [\-offset \fIinteger\fP] [\-xsymmetry] [\-ysymmetry] [\-erase\-mode \fIinteger\fP] [\-erase\-speed \fIusecs\fP] [\-delay \fIsecs\fP]
 .SH DESCRIPTION
 The \fIrorschach\fP program draws random patterns reminiscent of the
diff -ur xscreensaver-4.21.orig/hacks/rotor.man xscreensaver-4.21/hacks/rotor.man
--- xscreensaver-4.21.orig/hacks/rotor.man	2002-05-31 04:52:44.000000000 +0200
+++ xscreensaver-4.21/hacks/rotor.man	2005-05-17 17:10:17.000000000 +0200
@@ -1,8 +1,8 @@
 .TH XScreenSaver 1 "" "X Version 11"
 .SH NAME
-rotor - screen saver.
+rotor - screen saver
 .SH SYNOPSIS
-.B rotor
+.B /usr/lib/xscreensaver/rotor
 [\-display \fIhost:display.screen\fP]
 [\-visual \fIvisual\fP]
 [\-window]
diff -ur xscreensaver-4.21.orig/hacks/rotzoomer.man xscreensaver-4.21/hacks/rotzoomer.man
--- xscreensaver-4.21.orig/hacks/rotzoomer.man	2001-04-15 10:55:40.000000000 +0200
+++ xscreensaver-4.21/hacks/rotzoomer.man	2005-05-17 17:10:22.000000000 +0200
@@ -2,7 +2,7 @@
 .SH NAME
 rotzoomer - animated rotations and scalings of portions of the screen
 .SH SYNOPSIS
-.B rotzoomer
+.B /usr/lib/xscreensaver/rotzoomer
 [\-display \fIhost:display.screen\fP]
 [\-move | \-no\-move]
 [\-delay \fIusecs\fP]
diff -ur xscreensaver-4.21.orig/hacks/shadebobs.man xscreensaver-4.21/hacks/shadebobs.man
--- xscreensaver-4.21.orig/hacks/shadebobs.man	2003-06-29 00:24:24.000000000 +0200
+++ xscreensaver-4.21/hacks/shadebobs.man	2005-05-17 17:10:27.000000000 +0200
@@ -1,8 +1,8 @@
 .TH XScreenSaver 1 "" "X Version 11"
 .SH NAME
-shadebobs - oscillating vapor trails.
+shadebobs - oscillating vapor trails
 .SH SYNOPSIS
-.B shadebobs
+.B /usr/lib/xscreensaver/shadebobs
 [\-display \fIhost:display.screen\fP]
 [\-visual \fIvisual\fP]
 [\-window]
diff -ur xscreensaver-4.21.orig/hacks/sierpinski.man xscreensaver-4.21/hacks/sierpinski.man
--- xscreensaver-4.21.orig/hacks/sierpinski.man	1998-08-03 10:05:02.000000000 +0200
+++ xscreensaver-4.21/hacks/sierpinski.man	2005-05-17 17:10:31.000000000 +0200
@@ -2,7 +2,7 @@
 .SH NAME
 sierpinski - draws Sierpinski triangle fractals
 .SH SYNOPSIS
-.B sierpinski
+.B /usr/lib/xscreensaver/sierpinski
 [\-display \fIhost:display.screen\fP] [\-foreground \fIcolor\fP] [\-background \fIcolor\fP] [\-window] [\-root] [\-mono] [\-install] [\-visual \fIvisual\fP] [\-ncolors \fIinteger\fP] [\-delay \fImicroseconds\fP] [\-cycles \fIinteger\fP] [\-count \fIinteger\fP]
 
 .SH DESCRIPTION
diff -ur xscreensaver-4.21.orig/hacks/slidescreen.man xscreensaver-4.21/hacks/slidescreen.man
--- xscreensaver-4.21.orig/hacks/slidescreen.man	2001-04-15 10:55:40.000000000 +0200
+++ xscreensaver-4.21/hacks/slidescreen.man	2005-05-17 17:10:41.000000000 +0200
@@ -2,7 +2,7 @@
 .SH NAME
 slidescreen - permute the screen image like an 8-puzzle
 .SH SYNOPSIS
-.B slidescreen
+.B /usr/lib/xscreensaver/slidescreen
 [\-display \fIhost:display.screen\fP] [\-background \fIcolor\fP]
 [\-grid-size \fIpixels\fP] [\-ibw \fIpixels\fP] [\-increment \fIpixels\fP]
 [\-delay \fIusecs\fP] [\-delay2 \fIusecs\fP]
diff -ur xscreensaver-4.21.orig/hacks/slip.man xscreensaver-4.21/hacks/slip.man
--- xscreensaver-4.21.orig/hacks/slip.man	2001-04-15 10:55:40.000000000 +0200
+++ xscreensaver-4.21/hacks/slip.man	2005-05-17 17:10:44.000000000 +0200
@@ -2,7 +2,7 @@
 .SH NAME
 slip - sucks your screen into a jet engine
 .SH SYNOPSIS
-.B slip
+.B /usr/lib/xscreensaver/slip
 [\-display \fIhost:display.screen\fP] [\-foreground \fIcolor\fP]
 [\-background \fIcolor\fP] [\-window] [\-root] [\-mono] [\-install]
 [\-visual \fIvisual\fP] [\-ncolors \fIinteger\fP]
diff -ur xscreensaver-4.21.orig/hacks/sonar.man xscreensaver-4.21/hacks/sonar.man
--- xscreensaver-4.21.orig/hacks/sonar.man	2005-01-03 13:10:22.000000000 +0100
+++ xscreensaver-4.21/hacks/sonar.man	2005-05-17 17:11:39.000000000 +0200
@@ -11,11 +11,11 @@
 .if n .sp 1
 .if t .sp .5
 ..
-.TH Sonar 1 "3-Nov-98" "X Version 11"
+.TH XScreenSaver 1 "3-Nov-98" "X Version 11"
 .SH NAME
 sonar - display a sonar scope
 .SH SYNOPSIS
-.B sonar
+.B /usr/lib/xscreensaver/sonar
 [\-ping \fIhosts-or-subnets\fP]
 [\-background \fIcolor\fP]
 [\-sweep\-color \fIcolor\fP]
diff -ur xscreensaver-4.21.orig/hacks/speedmine.man xscreensaver-4.21/hacks/speedmine.man
--- xscreensaver-4.21.orig/hacks/speedmine.man	2001-05-19 07:16:45.000000000 +0200
+++ xscreensaver-4.21/hacks/speedmine.man	2005-05-17 17:11:02.000000000 +0200
@@ -15,7 +15,7 @@
 .SH NAME
 speedmine - simulates speeding down a rocky mineshaft, or a funky dancing worm
 .SH SYNOPSIS
-.B speedmine 
+.B /usr/lib/xscreensaver/speedmine 
 [\-display \fIhost:display.screen\fP] [\-root] [\-window]
 [\-install] [\-noinstall] [\-visual \fIvisual\fP] [\-wire] [\-nowire]
 [\-worm]
diff -ur xscreensaver-4.21.orig/hacks/sphere.man xscreensaver-4.21/hacks/sphere.man
--- xscreensaver-4.21.orig/hacks/sphere.man	1998-08-03 10:04:57.000000000 +0200
+++ xscreensaver-4.21/hacks/sphere.man	2005-05-17 17:11:18.000000000 +0200
@@ -2,7 +2,7 @@
 .SH NAME
 sphere - draws shaded spheres
 .SH SYNOPSIS
-.B sphere
+.B /usr/lib/xscreensaver/sphere
 [\-display \fIhost:display.screen\fP] [\-foreground \fIcolor\fP] [\-background \fIcolor\fP] [\-window] [\-root] [\-mono] [\-install] [\-visual \fIvisual\fP] [\-ncolors \fIinteger\fP] [\-delay \fImicroseconds\fP]
 
 .SH DESCRIPTION
diff -ur xscreensaver-4.21.orig/hacks/spiral.man xscreensaver-4.21/hacks/spiral.man
--- xscreensaver-4.21.orig/hacks/spiral.man	1998-08-03 10:05:09.000000000 +0200
+++ xscreensaver-4.21/hacks/spiral.man	2005-05-17 17:11:22.000000000 +0200
@@ -2,7 +2,7 @@
 .SH NAME
 spiral - draws moving circular spiral patterns
 .SH SYNOPSIS
-.B spiral
+.B /usr/lib/xscreensaver/spiral
 [\-display \fIhost:display.screen\fP] [\-foreground \fIcolor\fP] [\-background \fIcolor\fP] [\-window] [\-root] [\-mono] [\-install] [\-visual \fIvisual\fP] [\-ncolors \fIinteger\fP] [\-delay \fImicroseconds\fP] [\-count \fIinteger\fP]
 
 .SH DESCRIPTION
diff -ur xscreensaver-4.21.orig/hacks/spotlight.man xscreensaver-4.21/hacks/spotlight.man
--- xscreensaver-4.21.orig/hacks/spotlight.man	2001-04-15 10:55:40.000000000 +0200
+++ xscreensaver-4.21/hacks/spotlight.man	2005-05-17 17:11:50.000000000 +0200
@@ -2,7 +2,7 @@
 .SH NAME
 spotlight - move spotlight around desktop
 .SH SYNOPSIS
-.B spotlight
+.B /usr/lib/xscreensaver/spotlight
 [\-display \fIhost:display.screen\fP] [\-foreground \fIcolor\fP]
 [\-background \fIcolor\fP] [\-window] [\-root] [\-mono] [\-install]
 [\-visual \fIvisual\fP] [\-delay \fIusecs\fP] [\-radius \fIpixels\fP]
diff -ur xscreensaver-4.21.orig/hacks/squiral.man xscreensaver-4.21/hacks/squiral.man
--- xscreensaver-4.21.orig/hacks/squiral.man	1999-03-19 03:48:33.000000000 +0100
+++ xscreensaver-4.21/hacks/squiral.man	2005-05-17 17:28:25.000000000 +0200
@@ -1,8 +1,8 @@
-.TH XScreenSaver 1 "18-mar-1999" "X Version 11"
+.TH XScreenSaver 1 "18-Mar-1999" "X Version 11"
 .SH NAME
 squiral - square spirals screensaver
 .SH SYNOPSIS
-.B squiral
+.B /usr/lib/xscreensaver/squiral
 [\-display \fIhost:display.screen\fP] [\-foreground \fIcolor\fP]
 [\-background \fIcolor\fP] [\-window] [\-root] [\-mono] [\-install]
 [\-noinstall] [\-visual \fIvisual\fP] [\-fill \fIpercent\fP] [-count
diff -ur xscreensaver-4.21.orig/hacks/starfish.man xscreensaver-4.21/hacks/starfish.man
--- xscreensaver-4.21.orig/hacks/starfish.man	1998-08-03 10:05:01.000000000 +0200
+++ xscreensaver-4.21/hacks/starfish.man	2005-05-17 17:12:14.000000000 +0200
@@ -2,7 +2,7 @@
 .SH NAME
 starfish - radially-symmetric throbbing colormap-hacking graphics demo
 .SH SYNOPSIS
-.B starfish
+.B /usr/lib/xscreensaver/starfish
 [\-display \fIhost:display.screen\fP] [\-foreground \fIcolor\fP] [\-background \fIcolor\fP] [\-window] [\-root] [\-mono] [\-install] [\-visual \fIvisual\fP] [\-delay \fIusecs\fP] [\-delay2 \fIsecs\fP] [\-cycle\-delay2 \fIusecs\fP] [\-thickness \fIpixels\fP] [\-rotation \fIdegrees\fP] [\-duration \fIseconds\fP] [\-colors \fIint\fP] [\-cycle] [\-no\-cycle] [\-blob] [\-no\-blob]
 .SH DESCRIPTION
 The \fIstarfish\fP program draws radially symmetric objects, which expand,
diff -ur xscreensaver-4.21.orig/hacks/strange.man xscreensaver-4.21/hacks/strange.man
--- xscreensaver-4.21.orig/hacks/strange.man	1998-08-03 10:05:12.000000000 +0200
+++ xscreensaver-4.21/hacks/strange.man	2005-05-17 17:12:17.000000000 +0200
@@ -2,7 +2,7 @@
 .SH NAME
 strange - draws strange attractors
 .SH SYNOPSIS
-.B strange
+.B /usr/lib/xscreensaver/strange
 [\-display \fIhost:display.screen\fP] [\-foreground \fIcolor\fP] [\-background \fIcolor\fP] [\-window] [\-root] [\-mono] [\-install] [\-visual \fIvisual\fP] [\-ncolors \fIinteger\fP] [\-delay \fImicroseconds\fP]
 
 .SH DESCRIPTION
diff -ur xscreensaver-4.21.orig/hacks/substrate.man xscreensaver-4.21/hacks/substrate.man
--- xscreensaver-4.21.orig/hacks/substrate.man	2005-01-04 16:17:48.000000000 +0100
+++ xscreensaver-4.21/hacks/substrate.man	2005-05-17 17:13:14.000000000 +0200
@@ -1,8 +1,8 @@
-.TH Substrate 1 "08-Oct-04" "X Version 11"
+.TH XScreenSaver 1 "08-Oct-04" "X Version 11"
 .SH NAME
-substrate - Grow crystal-like lines on a computational substrate
+substrate - grow crystal-like lines on a computational substrate
 .SH SYNOPSIS
-.B substrate
+.B /usr/lib/xscreensaver/substrate
 [\-display \fIhost:display.screen\fP]
 [\-foreground \fIcolor\fP]
 [\-background \fIcolor\fP]
diff -ur xscreensaver-4.21.orig/hacks/swirl.man xscreensaver-4.21/hacks/swirl.man
--- xscreensaver-4.21.orig/hacks/swirl.man	2001-09-28 04:55:55.000000000 +0200
+++ xscreensaver-4.21/hacks/swirl.man	2005-05-17 17:12:25.000000000 +0200
@@ -2,7 +2,7 @@
 .SH NAME
 swirl - draws swirly color-cycling patterns
 .SH SYNOPSIS
-.B swirl
+.B /usr/lib/xscreensaver/swirl
 [\-display \fIhost:display.screen\fP] [\-foreground \fIcolor\fP]
 [\-background \fIcolor\fP] [\-window] [\-root] [\-mono] [\-install]
 [\-visual \fIvisual\fP]
diff -ur xscreensaver-4.21.orig/hacks/t3d.man xscreensaver-4.21/hacks/t3d.man
--- xscreensaver-4.21.orig/hacks/t3d.man	1999-01-10 04:40:15.000000000 +0100
+++ xscreensaver-4.21/hacks/t3d.man	2005-05-17 17:19:20.000000000 +0200
@@ -1,8 +1,9 @@
-.TH t3d 1 "Version 1.1" "Time 3D"
+.TH XScreenSaver 1
 .SH NAME
 t3d \- clock using flying balls to display the time
 .SH SYNOPSIS
-t3d [ \f2 options\f1 ]...
+.B /usr/lib/xscreensaver/t3d
+[ \f2 options\f1 ]...
 .SH DESCRIPTION
 .PP
 Time 3D is a clock. It uses flying balls to display the time. This
diff -ur xscreensaver-4.21.orig/hacks/thornbird.man xscreensaver-4.21/hacks/thornbird.man
--- xscreensaver-4.21.orig/hacks/thornbird.man	2002-05-31 05:09:06.000000000 +0200
+++ xscreensaver-4.21/hacks/thornbird.man	2005-05-17 17:26:56.000000000 +0200
@@ -1,8 +1,8 @@
 .TH XScreenSaver 1 "" "X Version 11"
 .SH NAME
-thornbird - Bird in a Thornbush fractal.
+thornbird - bird in a Thornbush fractal
 .SH SYNOPSIS
-.B thornbird
+.B /usr/lib/xscreensaver/thornbird
 [\-display \fIhost:display.screen\fP]
 [\-visual \fIvisual\fP]
 [\-window]
diff -ur xscreensaver-4.21.orig/hacks/triangle.man xscreensaver-4.21/hacks/triangle.man
--- xscreensaver-4.21.orig/hacks/triangle.man	2002-05-31 04:50:14.000000000 +0200
+++ xscreensaver-4.21/hacks/triangle.man	2005-05-17 17:14:24.000000000 +0200
@@ -1,8 +1,8 @@
 .TH XScreenSaver 1 "" "X Version 11"
 .SH NAME
-triangle - random mountains using iterative subdivision of triangles.
+triangle - random mountains using iterative subdivision of triangles
 .SH SYNOPSIS
-.B triangle
+.B /usr/lib/xscreensaver/triangle
 [\-display \fIhost:display.screen\fP]
 [\-visual \fIvisual\fP]
 [\-window]
diff -ur xscreensaver-4.21.orig/hacks/truchet.man xscreensaver-4.21/hacks/truchet.man
--- xscreensaver-4.21.orig/hacks/truchet.man	1998-06-06 05:14:13.000000000 +0200
+++ xscreensaver-4.21/hacks/truchet.man	2005-05-17 17:19:29.000000000 +0200
@@ -2,7 +2,7 @@
 .SH NAME
 truchet - draws curved or angular Truchet patterns
 .SH SYNOPSIS
-.B truchet
+.B /usr/lib/xscreensaver/truchet
 [\-display \fIhost:display.screen\fP]  [\-foreground \fIcolor\fP] [\-background \fIcolor\fP]
 [\-window] [\-root] [\-mono] [\-install] [\-visual \fIvisual\fP] [\-delay \fImicroseconds\fP]
 [\-min\-width \fIinteger\fP] [\-min-height \fIinteger\fP] [\-max-width \fIinteger\fP] 
diff -ur xscreensaver-4.21.orig/hacks/twang.man xscreensaver-4.21/hacks/twang.man
--- xscreensaver-4.21.orig/hacks/twang.man	2002-02-09 20:12:35.000000000 +0100
+++ xscreensaver-4.21/hacks/twang.man	2005-05-17 17:14:39.000000000 +0200
@@ -2,7 +2,7 @@
 .SH NAME
 twang - pluck pieces of the screen
 .SH SYNOPSIS
-.B twang
+.B /usr/lib/xscreensaver/twang
 [\-display \fIhost:display.screen\fP] [\-foreground \fIcolor\fP] [\-background \fIcolor\fP] [\-window] [\-root] [\-mono] [\-install] [\-visual \fIvisual\fP] [\-shm] [\-no-shm] [\-delay \fImicroseconds\fP] [\-border-color \fIcolor\fP] [\-border-width \fIinteger\fP] [\-event-chance \fIfraction\fP] [\-friction \fIfraction\fP] [\-springiness \fIfraction\fP] [\-tile-size \fIinteger\fP] [\-transference \fIfraction\fP]
 .SH DESCRIPTION
 \fITwang\fP divides the screen into equal-sized tiles, and then plucks
diff -ur xscreensaver-4.21.orig/hacks/vermiculate.man xscreensaver-4.21/hacks/vermiculate.man
--- xscreensaver-4.21.orig/hacks/vermiculate.man	2002-05-31 05:06:40.000000000 +0200
+++ xscreensaver-4.21/hacks/vermiculate.man	2005-05-17 17:14:45.000000000 +0200
@@ -1,8 +1,8 @@
 .TH XScreenSaver 1 "" "X Version 11"
 .SH NAME
-vermiculate - to move in a worm-like manner.
+vermiculate - to move in a worm-like manner
 .SH SYNOPSIS
-.B vermiculate
+.B /usr/lib/xscreensaver/vermiculate
 [\-display \fIhost:display.screen\fP]
 [\-visual \fIvisual\fP]
 [\-window]
diff -ur xscreensaver-4.21.orig/hacks/vidwhacker.man xscreensaver-4.21/hacks/vidwhacker.man
--- xscreensaver-4.21.orig/hacks/vidwhacker.man	2002-11-20 09:20:02.000000000 +0100
+++ xscreensaver-4.21/hacks/vidwhacker.man	2005-05-17 17:14:49.000000000 +0200
@@ -2,7 +2,7 @@
 .SH NAME
 vidwhacker - grab images and apply random filters to them
 .SH SYNOPSIS
-.B vidwhacker
+.B /usr/lib/xscreensaver/vidwhacker
 [\-display \fIhost:display.screen\fP] [\-root] [\-verbose]
 [\-stdin] [\-stdout] [\-delay seconds]
 [-directory \fIdirectory\fP]
diff -ur xscreensaver-4.21.orig/hacks/vines.man xscreensaver-4.21/hacks/vines.man
--- xscreensaver-4.21.orig/hacks/vines.man	1998-08-03 10:05:03.000000000 +0200
+++ xscreensaver-4.21/hacks/vines.man	2005-05-17 17:14:53.000000000 +0200
@@ -2,7 +2,7 @@
 .SH NAME
 vines - draws pseudo-fractal geometric patterns
 .SH SYNOPSIS
-.B vines
+.B /usr/lib/xscreensaver/vines
 [\-display \fIhost:display.screen\fP] [\-foreground \fIcolor\fP] [\-background \fIcolor\fP] [\-window] [\-root] [\-mono] [\-install] [\-visual \fIvisual\fP] [\-ncolors \fIinteger\fP] [\-delay \fImicroseconds\fP]
 
 .SH DESCRIPTION
diff -ur xscreensaver-4.21.orig/hacks/wander.man xscreensaver-4.21/hacks/wander.man
--- xscreensaver-4.21.orig/hacks/wander.man	2002-05-31 05:04:05.000000000 +0200
+++ xscreensaver-4.21/hacks/wander.man	2005-05-17 17:15:10.000000000 +0200
@@ -1,8 +1,8 @@
 .TH XScreenSaver 1 "" "X Version 11"
 .SH NAME
-wander - colorful random-walk.
+wander - colorful random-walk
 .SH SYNOPSIS
-.B wander
+.B /usr/lib/xscreensaver/wander
 [\-display \fIhost:display.screen\fP]
 [\-visual \fIvisual\fP]
 [\-window]
diff -ur xscreensaver-4.21.orig/hacks/webcollage.man xscreensaver-4.21/hacks/webcollage.man
--- xscreensaver-4.21.orig/hacks/webcollage.man	2005-01-19 11:32:52.000000000 +0100
+++ xscreensaver-4.21/hacks/webcollage.man	2005-05-17 17:15:16.000000000 +0200
@@ -15,7 +15,7 @@
 .SH NAME
 webcollage - decorate the screen with random images from the web
 .SH SYNOPSIS
-.B webcollage
+.B /usr/lib/xscreensaver/webcollage
 [\-display \fIhost:display.screen\fP]
 [\-root]
 [\-window\-id \fIid\fP]
diff -ur xscreensaver-4.21.orig/hacks/whirlwindwarp.man xscreensaver-4.21/hacks/whirlwindwarp.man
--- xscreensaver-4.21.orig/hacks/whirlwindwarp.man	2001-01-20 05:45:33.000000000 +0100
+++ xscreensaver-4.21/hacks/whirlwindwarp.man	2005-05-17 17:28:21.000000000 +0200
@@ -1,8 +1,8 @@
-.TH XScreenSaver 1 "13-aug-92" "X Version 11"
+.TH XScreenSaver 1 "13-Aug-92" "X Version 11"
 .SH NAME
 whirlwindwarp - crazy moving stars
 .SH SYNOPSIS
-.B whirlwindwarp
+.B /usr/lib/xscreensaver/whirlwindwarp
 [\-display \fIhost:display.screen\fP] [\-foreground \fIcolor\fP] [\-background \fIcolor\fP] [\-window] [\-root] [\-mono] [\-install] [\-visual \fIvisual\fP] [\-points \fIinteger\fP] [\-tails \fIinteger\fP]
 .SH DESCRIPTION
 \fIwhirlwindwarp\fP plots stars moving according to various forcefields
diff -ur xscreensaver-4.21.orig/hacks/whirlygig.man xscreensaver-4.21/hacks/whirlygig.man
--- xscreensaver-4.21.orig/hacks/whirlygig.man	2002-11-20 09:01:39.000000000 +0100
+++ xscreensaver-4.21/hacks/whirlygig.man	2005-05-17 17:15:28.000000000 +0200
@@ -1,8 +1,8 @@
 .TH XScreenSaver 1 "31-Mar-01" "X Version 11"
 .SH NAME
-whirlygig -- zooming chains of sinusoidal spots
+whirlygig - zooming chains of sinusoidal spots
 .SH SYNOPSIS
-.B whirlygig
+.B /usr/lib/xscreensaver/whirlygig
 [\-display \fIhost:display.screen\fP] [\-window] [\-root] [\-mono]
 [\-install] [\-noinstall] [\-visual arg] [\-window-id arg]   
 [\-xspeed arg] [\-yspeed arg] [\-whirlies arg] [\-nlines arg] 
diff -ur xscreensaver-4.21.orig/hacks/worm.man xscreensaver-4.21/hacks/worm.man
--- xscreensaver-4.21.orig/hacks/worm.man	2002-05-31 04:52:00.000000000 +0200
+++ xscreensaver-4.21/hacks/worm.man	2005-05-17 17:25:23.000000000 +0200
@@ -1,8 +1,8 @@
 .TH XScreenSaver 1 "" "X Version 11"
 .SH NAME
-worm - multicolored worms that crawl around the screen.
+worm - multicolored worms that crawl around the screen
 .SH SYNOPSIS
-.B worm
+.B /usr/lib/xscreensaver/worm
 [\-display \fIhost:display.screen\fP]
 [\-visual \fIvisual\fP]
 [\-window]
diff -ur xscreensaver-4.21.orig/hacks/wormhole.man xscreensaver-4.21/hacks/wormhole.man
--- xscreensaver-4.21.orig/hacks/wormhole.man	2004-02-12 02:11:29.000000000 +0100
+++ xscreensaver-4.21/hacks/wormhole.man	2005-05-17 17:28:18.000000000 +0200
@@ -1,8 +1,8 @@
-.TH XScreenSaver 1 "11-feb-04" "X Version 11"
+.TH XScreenSaver 1 "11-Feb-04" "X Version 11"
 .SH NAME
 wormhole - animation of flying through a wormhole
 .SH SYNOPSIS
-.B wormhole
+.B /usr/lib/xscreensaver/wormhole
 [\-display \fIhost:display.screen\fP]
 [\-window]
 [\-visual \fIvisual\fP]
diff -ur xscreensaver-4.21.orig/hacks/xanalogtv.man xscreensaver-4.21/hacks/xanalogtv.man
--- xscreensaver-4.21.orig/hacks/xanalogtv.man	2003-10-25 07:42:44.000000000 +0200
+++ xscreensaver-4.21/hacks/xanalogtv.man	2005-05-17 17:15:50.000000000 +0200
@@ -1,8 +1,8 @@
 .TH XScreenSaver 1 "10-Oct-03" "X Version 11"
 .SH NAME
-xanalogtv - Simulate reception on an old analog TV set
+xanalogtv - simulate reception on an old analog TV set
 .SH SYNOPSIS
-.B xanalogtv
+.B /usr/lib/xscreensaver/xanalogtv
 [\-display \fIhost:display.screen\fP] [\-window] [\-root] [\-install]
 [\-visual \fIvisual\fP] 
 [\-cycle] [\-no-cycle]
diff -ur xscreensaver-4.21.orig/hacks/xflame.man xscreensaver-4.21/hacks/xflame.man
--- xscreensaver-4.21.orig/hacks/xflame.man	2003-10-06 07:21:57.000000000 +0200
+++ xscreensaver-4.21/hacks/xflame.man	2005-05-17 17:15:54.000000000 +0200
@@ -2,7 +2,7 @@
 .SH NAME
 xflame - draws animated flames
 .SH SYNOPSIS
-.B xflame
+.B /usr/lib/xscreensaver/xflame
 [\-display \fIhost:display.screen\fP] [\-window] [\-root] [\-install]
 [\-visual \fIvisual\fP]
 [\-foreground \fIcolor\fP]
diff -ur xscreensaver-4.21.orig/hacks/xjack.man xscreensaver-4.21/hacks/xjack.man
--- xscreensaver-4.21.orig/hacks/xjack.man	1998-08-03 10:04:55.000000000 +0200
+++ xscreensaver-4.21/hacks/xjack.man	2005-05-17 17:28:13.000000000 +0200
@@ -1,8 +1,8 @@
-.TH XScreenSaver 1 "18-sep-97" "X Version 11"
+.TH XScreenSaver 1 "18-Sep-97" "X Version 11"
 .SH NAME
 xjack - all work and no play makes jack a dull boy
 .SH SYNOPSIS
-.B xjack
+.B /usr/lib/xscreensaver/xjack
 [\-display \fIhost:display.screen\fP] [\-foreground \fIcolor\fP] [\-background \fIcolor\fP] [\-window] [\-root] [\-delay \fIusecs\fP]
 .SH DESCRIPTION
 All work and no play makes jack a dull boy.  All work and no play makes jack
diff -ur xscreensaver-4.21.orig/hacks/xlyap.man xscreensaver-4.21/hacks/xlyap.man
--- xscreensaver-4.21.orig/hacks/xlyap.man	1997-11-26 00:43:36.000000000 +0100
+++ xscreensaver-4.21/hacks/xlyap.man	2005-05-17 17:18:08.000000000 +0200
@@ -1,11 +1,11 @@
-.TH XLYAP 6X
+.TH XScreenSaver 1
 .SH NAME
 xlyap \- display an array of Lyapunov exponents graphically
 .SH SYNOPSIS
 .in +8n
 .ti -8n
-\fIxlyap\fR
-[-BLps][-W width][-H height][-o filename][-a 
+.B /usr/lib/xscreensaver/xlyap
+[-BLps] [-W width] [-H height] [-o filename] [-a 
 \fIn\fR ]
 [-b 
 \fIn\fR ]
@@ -22,7 +22,7 @@
 \fIn\fR ]
 [-D 
 \fIn\fR ]
-[-F string][-f string][-r 
+[-F string] [-f string] [-r 
 \fIn\fR ]
 [-O 
 \fIn\fR ]
diff -ur xscreensaver-4.21.orig/hacks/xmatrix.man xscreensaver-4.21/hacks/xmatrix.man
--- xscreensaver-4.21.orig/hacks/xmatrix.man	2003-06-13 03:33:41.000000000 +0200
+++ xscreensaver-4.21/hacks/xmatrix.man	2005-05-17 17:18:15.000000000 +0200
@@ -2,7 +2,7 @@
 .SH NAME
 xmatrix - simulates the computer displays from the movie
 .SH SYNOPSIS
-.B xmatrix
+.B /usr/lib/xscreensaver/xmatrix
 [\-display \fIhost:display.screen\fP] [\-window] [\-root] [\-install]
 [\-visual \fIvisual\fP] 
 [\-delay \fIusecs\fP] 
diff -ur xscreensaver-4.21.orig/hacks/xrayswarm.man xscreensaver-4.21/hacks/xrayswarm.man
--- xscreensaver-4.21.orig/hacks/xrayswarm.man	2002-05-31 05:06:10.000000000 +0200
+++ xscreensaver-4.21/hacks/xrayswarm.man	2005-05-17 17:26:59.000000000 +0200
@@ -1,8 +1,8 @@
 .TH XScreenSaver 1 "" "X Version 11"
 .SH NAME
-xrayswarm - swarms with color trails.
+xrayswarm - swarms with color trails
 .SH SYNOPSIS
-.B xrayswarm
+.B /usr/lib/xscreensaver/xrayswarm
 [\-display \fIhost:display.screen\fP]
 [\-visual \fIvisual\fP]
 [\-window]
diff -ur xscreensaver-4.21.orig/hacks/xspirograph.man xscreensaver-4.21/hacks/xspirograph.man
--- xscreensaver-4.21.orig/hacks/xspirograph.man	2002-02-12 04:33:24.000000000 +0100
+++ xscreensaver-4.21/hacks/xspirograph.man	2005-05-17 17:18:36.000000000 +0200
@@ -2,7 +2,7 @@
 .SH NAME
 xspirograph - simulates the rotation of a disk inside a circular rim
 .SH SYNOPSIS
-.B xspirograph
+.B /usr/lib/xscreensaver/xspirograph
 [\-display \fIhost:display.screen\fP] [\-window] [\-root] [\-install]
 [\-visual \fIvisual\fP] 
 [\-delay \fIseconds\fP] 
diff -ur xscreensaver-4.21.orig/hacks/xsublim.man xscreensaver-4.21/hacks/xsublim.man
--- xscreensaver-4.21.orig/hacks/xsublim.man	2000-07-17 06:25:27.000000000 +0200
+++ xscreensaver-4.21/hacks/xsublim.man	2005-05-17 17:26:15.000000000 +0200
@@ -1,8 +1,8 @@
-.TH XSublim 1 "16-Jul-99" "X Version 11"
+.TH XScreenSaver 1 "16-Jul-99" "X Version 11"
 .SH NAME
-xsublim - Display (submit) "subliminal" (conform) messages (obey)
+xsublim - display (submit) "subliminal" (conform) messages (obey)
 .SH SYNOPSIS
-.B xsublim
+.B /usr/lib/xscreensaver/xsublim
 [\-display \fIhost:display.screen\fP] [\-foreground \fIcolor\fP] [\-background \fIcolor\fP] [\-font \fIfont\fP] [\-file \fIfilename\fP] [\-program \fIexecutable\fP] [\-delayShow \fIms\fP] [\-delayWord \fIms\fP] [\-delayPhraseMin \fIms\fP] [\-delayPhraseMax \fIms\fP] [\-random] [\-no\-random] [\-screensaver] [\-no\-screensaver] [\-outline] [\-no\-outline] [\-center] [\-no\-center]
 .SH DESCRIPTION
 The \fIxsublim\fP program quickly (consume) draws and erases inspirational
diff -ur xscreensaver-4.21.orig/hacks/xteevee.man xscreensaver-4.21/hacks/xteevee.man
--- xscreensaver-4.21.orig/hacks/xteevee.man	2003-10-14 05:11:46.000000000 +0200
+++ xscreensaver-4.21/hacks/xteevee.man	2005-05-17 17:18:50.000000000 +0200
@@ -1,8 +1,8 @@
 .TH XScreenSaver 1 "19-Nov-99" "X Version 11"
 .SH NAME
-xteevee - Simulate various TV problems
+xteevee - simulate various TV problems
 .SH SYNOPSIS
-.B xteevee
+.B /usr/lib/xscreensaver/xteevee
 [\-display \fIhost:display.screen\fP] [\-window] [\-root] [\-install]
 [\-visual \fIvisual\fP] 
 [\-cycle] [\-no-cycle]
diff -ur xscreensaver-4.21.orig/hacks/zoom.man xscreensaver-4.21/hacks/zoom.man
--- xscreensaver-4.21.orig/hacks/zoom.man	2001-04-15 10:55:41.000000000 +0200
+++ xscreensaver-4.21/hacks/zoom.man	2005-05-17 17:18:57.000000000 +0200
@@ -2,7 +2,7 @@
 .SH NAME
 zoom - wander around magified desktop
 .SH SYNOPSIS
-.B zoom
+.B /usr/lib/xscreensaver/zoom
 [\-display \fIhost:display.screen\fP] [\-foreground \fIcolor\fP]
 [\-background \fIcolor\fP] [\-window] [\-root] [\-mono] [\-install]
 [\-visual \fIvisual\fP] [\-delay \fIusecs\fP] [\-lenses]
