diff -ru xscreensaver-3.34/configure.in xscreensaver-3.34-AX/configure.in
--- xscreensaver-3.34/configure.in	Tue Oct 23 07:13:30 2001
+++ xscreensaver-3.34-AX/configure.in	Thu Nov 22 23:17:50 2001
@@ -2139,10 +2139,8 @@
          CPPFLAGS="$CPPFLAGS $X_CFLAGS"
 
           # M4 sucks!!
-         changequote(X,Y)
          mglv=`(eval "$ac_cpp conftest.$ac_ext") 2>&AC_FD_CC | sed -n \
-              's/^configure:.*\([0-9][0-9]*\).*\([0-9][0-9]*\).*$/\1.\2/p'`
-         changequote([,])
+              ['s/^configure:.*\([0-9][0-9]*\).*\([0-9][0-9]*\).*$/\1.\2/p']`
 
          rm -f conftest.$ac_ext
 
