diff -urN aa/xscreensaver-4.15/configure.in xscreensaver-4.15/configure.in
--- aa/xscreensaver-4.15/configure.in	2004-02-20 23:58:44.000000000 +0100
+++ xscreensaver-4.15/configure.in	2004-02-28 00:20:21.019623224 +0100
@@ -874,7 +874,7 @@
 AC_DEFINE_UNQUOTED(PACKAGE, "$GETTEXT_PACKAGE")
 AC_SUBST(GETTEXT_PACKAGE)
 
-ALL_LINGUAS="ca da de es et fi fr hu it ja ko nb nl no pl pt pt_BR ru sk sv vi wa zh_CN zh_TW"
+ALL_LINGUAS="ca da de es et fi fr hu it ja ko nb nl pl pt pt_BR ru sk sv vi wa zh_CN zh_TW"
 AM_GLIB_GNU_GETTEXT
 MKINSTALLDIRS="$INSTALL_DIRS"
 
