diff -aurN xscreensaver-4.21.orig/driver/screensaver-properties.desktop.in xscreensaver-4.21/driver/screensaver-properties.desktop.in
--- xscreensaver-4.21.orig/driver/screensaver-properties.desktop.in	2003-12-15 11:21:33.000000000 +0100
+++ xscreensaver-4.21/driver/screensaver-properties.desktop.in	2005-04-19 12:35:02.000000000 +0200
@@ -1,8 +1,9 @@
 [Desktop Entry]
+Encoding=UTF-8
 Exec=xscreensaver-demo
-Icon=xscreensaver.xpm
+Icon=screensaver
 Terminal=false
 _Name=Screensaver
 _Comment=Change screensaver properties
 Type=Application
-Categories=Application;Settings;Appearance;AdvancedSettings;
+Categories=GTK;GNOME;Settings;Appearance;AdvancedSettings;X-Capplet;
