diff -urN xscreensaver-4.03.orig/driver/Makefile.in xscreensaver-4.03/driver/Makefile.in
--- xscreensaver-4.03.orig/driver/Makefile.in	Sun Sep  1 00:47:49 2002
+++ xscreensaver-4.03/driver/Makefile.in	Sun Sep  1 00:48:27 2002
@@ -27,7 +27,7 @@
 GNOME_PANELDIR	= $(GNOME_DATADIR)/gnome/apps/Settings/Desktop
 GNOME_BINDIR	= $(bindir)
 HACK_CONF_DIR	= @HACK_CONF_DIR@
-GTK_ICONDIR	= @GTK_ICONDIR@
+GTK_ICONDIR	= /usr/X11R6/share/pixmaps
 
 CC		= @CC@
 CFLAGS		= @CFLAGS@
